Content: Remnant Chapter 2 Cognizance part II (#6392)
endless-sky/endless-sky: data/remnant/remnant 2 cognizance.txt

	on complete
		conversation
			`Disappointed in the void sprites' behavior, you deliver the recordings to Plume's lab without fanfare. Plume quickly takes and reviews the footage before making a gesture that you think means 'curious dejection.' Then he seems to notice something, and resumes his usual intent bearing.`
			`	"Here, look!" He rewinds the recording and gestures curtly at the projection in front of you. "There. That one is injured. Some kind of burn in a long, straight line." He zips through the footage farther: "And here, a scattering of scorch marks." He looks up at you and snaps out a series of gestures: "This is outside my expertise; I will need to consult an expert. Meet me in the spaceport in an hour."`

mission "Remnant: Cognizance 2"
	name "Eliminate Korath Ship"
	description "Plume ordered you to eliminate the Korath ship in the <waypoints> system."
	source "Aventine"
	waypoint
		system "Nenia"
	to offer
		has "Remnant: Cognizance 1: done"
	on offer
		conversation

> 			`You arrive in the spaceport to find Plume pacing quickly along a walkway. He gestures a quick greeting and jumps straight to the point. "We have never seen marks like these naturally appear on void sprites, but they match the weapon scoring patterns we have seen delivered by Korath ships. As such, we believe the Korath have begun hunting void sprites."`

Author: Scaring isn't the word I'm going for here.

Scoring, as in damaging, marking. From Mirriam-Webster:
> to mark with or as if with a line or groove

    the glassblower scored the glass rod first so that it would break cleanly

	on offer
		conversation
			`As you finish your landing checks, your commlink notifies you of an incoming message from Dawn:`
			`	"Greetings, Captain <first>! We will be compiling updated navigational charts for the new systems and would like a fresh copy of the data your nav system recorded. We are hoping to retrieve some additional data that is not routinely downloaded by the automated starport systems. Stop by <planet>, and the starport systems will copy the data during the systems check. It's a pretty small data set, so there's no need to use data crystals. Thank you!"`
			`	The end of the message is the location of a starport on <planet>, presumably where you are supposed to go.`
				accept
	on complete
		payment 100000
		event "remnant: cognizance 1 systems available"
		dialog `As the starport computer downloads the data, your computer displays a message tacked onto the log file: "Thank you for providing the additional data from your navigational logs. It will be useful to contrast against those of our scouts. A logistics adjustment of <payment> has been authorized for the <ship>."`

mission "Remnant: Cognizance 20"
	name "Preparation for expedition"

> 	description "Outfit a research laboratory and go to <planet>."

Reviewer: ```suggestion
	description "Outfit a research laboratory, retrieve equipment from <stopovers>, then bring it to <destination>."
```
Not knowing about the stopover can make the mission confusing.

			`	"So they banded together. It was not, I gather from the records, an unusual thing for academics of a particular field to meet with others of their field. If one faculty could not fund an expedition, perhaps two different faculties working together could. Eventually they formed a loose organization, a sort of 'standing conference' on astronomy and astrophysics, through which they met to discuss, debate, and organize."`
			choice
				`	"Your ancestors were originally a scientific conference?"`
				`	"Thanks for the background. So, what is our mission?"`
					goto missiontransition
			`	Several of the Remnant nod, and another signs. "They gathered together to talk of their science, and each brought with them the captains and crew who enabled them. Some of those captains brought with them resources, too. Who else would appreciate learning of the void as much as those who ply their trades across it?`

> 			`	"The association grew over time, although as best we can tell from our records it made an effort to remain both politically and economically neutral. As such, it remained comparatively small and close-knit. Many of our ancestors have told stories about calling themselves the 'Exploration Vector,' alluding to the fact that they were the one subset of humanity on a course to head out into the furthest reaches of space. They even held contests about who could go the furthest from inhabited space, or who could spend the longest time between resupply stops.`

Author: *innocent look* Maaaaybe. ;)

			`	"When the war against the Alphas started, it was these people who had the most knowledge and experience with the farthest reaches of space. While many outfitters might sell a captain a map showing the hyperjumps a few systems into the uninhabited territories, it was our ancestors who had spent years living in those systems while they did their research. Thus they were the ones who got the jobs running supplies, or flying reconnaissance. They saw first-hand the way the war was going. They made the rendezvous' and picked up the equipment drops, and they grew increasingly worried."`
			choice
				`	"They were worried about the course the war was taking?"`
				`	"That would be worrisome. We should focus on the mission."`
					goto missiontransition

> 			`	Dust starts gesturing again. "Yes, but also by what they saw. Strange ships that slipped over the horizon, odd energy signatures in the distance, novel equipment that simply showed up on Deep ships without explanation... It seemed like someone out there was taking advantage of the chaos to influence humanity. Someone beyond the reach of the normal hyperspace network. It spurred our ancestors to revisit old theories and kindle new ones. Knowing that there was someone, something beyond what we knew made us bold. It was not long before the first ship travelled through the wormhole."`

Reviewer: ```suggestion
			`	Dusk starts gesturing again. "Yes, but also by what they saw. Strange ships that slipped over the horizon, odd energy signatures in the distance, novel equipment that simply showed up on Deep ships without explanation... It seemed like someone out there was taking advantage of the chaos to influence humanity. Someone beyond the reach of the normal hyperspace network. It spurred our ancestors to revisit old theories and kindle new ones. Knowing that there was someone, something beyond what we knew made us bold. It was not long before the first ship travelled through the wormhole."`
```

> 			`	Dusk starts gesturing again. "Yes, but also by what they saw. Strange ships that slipped over the horizon, odd energy signatures in the distance, novel equipment that simply showed up on Deep ships without explanation... It seemed like someone out there was taking advantage of the chaos to influence humanity. Someone beyond the reach of the normal hyperspace network. It spurred our ancestors to revisit old theories and kindle new ones. Knowing that there was someone, something beyond what we knew made us bold. It was not long before the first ship travelled through the wormhole."`

Reviewer: ```suggestion
			`	Dusk starts gesturing again. "Yes, but also by what they saw. Strange ships that slipped over the horizon, odd energy signatures in the distance, novel equipment that simply showed up on Deep ships without explanation... It seemed like someone out there was taking advantage of the chaos to influence humanity. Someone beyond the reach of the normal hyperspace network. It spurred our ancestors to revisit old theories and kindle new ones. Knowing that there was someone, something beyond what we knew made us bold. It was not long before the first ship traveled through the wormhole."`
```

			choice
				`	"Which ship was that?"`
				`	"I think I can see where this is going. Time to get to work."`
					goto missiontransition

> 			`	Dust smiles "The Shadow Step, an independent science vessel who did double-duty as a gas miner. She made a quiet fortune harvesting rarer gases from obscure gas planets off the charted paths. Never enough to attract the attention of the big corporate entities, but enough to be comfortable. Her captain was always making modifications. Not a genius by any means, but what they lacked in brilliance, they made up for in dedication and willingness to experiment. According to our records, making modifications to the Shadow Step was their main hobby, and the small crew she flew with had similar interests." Dust pauses to take a drink.`

Reviewer: ```suggestion
			`	Dusk smiles "The Shadow Step, an independent science vessel who did double-duty as a gas miner. She made a quiet fortune harvesting rarer gases from obscure gas planets off the charted paths. Never enough to attract the attention of the big corporate entities, but enough to be comfortable. Her captain was always making modifications. Not a genius by any means, but what they lacked in brilliance, they made up for in dedication and willingness to experiment. According to our records, making modifications to the Shadow Step was their main hobby, and the small crew she flew with had similar interests." Dusk pauses to take a drink.`
```

> 			`	"Thanks to the bravery of the Shadow Step and her crew, the hard work of the researchers that worked with them, and the contributions of a vast number of other ships and teams, the way out was discovered." Dust signs this last with a note of finality, and many of the Remnant who were listening make the sign for 'respect' and 'together.'`

Reviewer: ```suggestion
			`	"Thanks to the bravery of the Shadow Step and her crew, the hard work of the researchers that worked with them, and the contributions of a vast number of other ships and teams, the way out was discovered." Dusk signs this last with a note of finality, and many of the Remnant who were listening make the sign for 'respect' and 'together.'`
```

			label missiontransition

> 			`	Dusk smiles and signs apologetically. "My apologies for being overly detailed. I found the study of our ancestors to be an intriguing one, so I have spent much time searching out my families' history. That, in turn, has led to a detailed interest in the early years of our society." Dust gestures towards the holographic projection of Ssil Vida. "But enough of history, the present calls to us."`

Reviewer: ```suggestion
			`	Dusk smiles and signs apologetically. "My apologies for being overly detailed. I found the study of our ancestors to be an intriguing one, so I have spent much time searching out my families' history. That, in turn, has led to a detailed interest in the early years of our society." Dusk gestures towards the holographic projection of Ssil Vida. "But enough of history, the present calls to us."`
```

			label missiondescription

> 			`	Dust leans over the holo, peering at the details. "The mission is a very standard arc: We go to the station, set up our equipment to get some detailed readings, and return here once we are done. For these initial readings we should need about a day onsite, and we will leave some of our equipment to take ongoing recordings for retrieval later. After that... Well, we will see what the data says and decide from there."`

Reviewer: ```suggestion
			`	Dusk leans over the holo, peering at the details. "The mission is a very standard arc: We go to the station, set up our equipment to get some detailed readings, and return here once we are done. For these initial readings we should need about a day onsite, and we will leave some of our equipment to take ongoing recordings for retrieval later. After that... Well, we will see what the data says and decide from there."`
```

				accept
	on stopover
		require "Research Laboratory"
		conversation
			`Dusk and the team of scientists waste no time in getting to work: They have the preliminary sensors working almost from the moment you left the spaceport, and by the time you dock with Ssil Vida they have all their equipment arranged to quickly offload and setup. With the space cleared, they expand the research laboratory and set to work analyzing the system. Before long they are clustered around displays and adjusting sensor readings, deep in discussion. Eventually, Dusk strides over to you. "Everything is setup and proceeding nominally, Captain. We will need a few hours to complete the measurements, you do not need to stand around watching, although you are welcome to if you like." Dusk heads back over to check on one of the devices.`
			choice

> 				`	Return to the <ship> and prepare for departure.`
> 					goto relax
> 				`	Ask if you can help with their work.`
> 					goto scienceassistant
> 				`	Explore Ssil Vida.`

Reviewer: ```suggestion
				`	(Return to the <ship> and prepare for departure.)`
					goto relax
				`	(Ask if you can help with their work.)`
					goto scienceassistant
				`	(Explore Ssil Vida.)`
```

			apply explorationinterest += 1
			`	You pick up some instruments, recording equipment, and wayfinding labels, then check the map tracking the other Remnant researchers. Picking an unexplored direction, you set off.`

> 			`	The first few corridors are marked with labels indicating the direction that other explorers traveled. Picking an unmarked hallway, you put up a wayfinding label of your own, and set off along it. Almost immediately the sense of scale you noticed in the main room is reinforced, as you travel considerable distance between doorways. Glancing inside, you find room after room containing large platforms. Examining them, you find that each platform is lightly padded with several bars set into it along each side. At each end of the platform the surface has what you would guess to be storage lockers embedded in it. Based on your experience, you realize that these resemble storage platforms used to secure delicate cargo, but there is no indication of what that cargo might be.`

Author: I'm not really sure how to change it. It is intended to convey actual information about exploring a small piece of Ssil Vida, and allow the player to experience that exploration. Replacing it with something short and to the point would lose the impact. Or at least, the re-writing that comes to mind does so. "You explore a small section of the station and discover that the oversized nature extends everywhere." just doesn't cut it. I want players to feel like they are actually there, wandering down unknown hallways, peering into rooms no human has ever seen before.

> 			`	The first few corridors are marked with labels indicating the direction that other explorers traveled. Picking an unmarked hallway, you put up a wayfinding label of your own, and set off along it. Almost immediately the sense of scale you noticed in the main room is reinforced, as you travel considerable distance between doorways. Glancing inside, you find room after room containing large platforms. Examining them, you find that each platform is lightly padded with several bars set into it along each side. At each end of the platform the surface has what you would guess to be storage lockers embedded in it. Based on your experience, you realize that these resemble storage platforms used to secure delicate cargo, but there is no indication of what that cargo might be.`

Author: @TheMarksman-ES  OK, a few days away from it and I've done some edits to break up the step by step tell nature of it a little. I think it is better now.

			choice
				`	The surface vault.`
					goto surfacevault
				`	The deep vault.`
			`	The Remnant leads you through several sets of increasingly impressive doors before finally taking you through an almost invisible door tucked in the back corner of a storage room. Inside, he pauses and gestures "I would recommend you hold onto the rails, and either hook your feet onto the toeholds or activate the magnets in your boots, if you have them. The drop is quite sudden." Looking around, you note that there are what look like toeholds bolted to the floor around the edge of the room, and your guide has inserted his feet into one pair.`
			`	As soon as you gesture that you are ready, the floor seems to drop out from below you. It seems as though you are in an elevator dropping down so fast that you are experiencing negative g forces. After several minutes it begins to slow down and instead of feeling lighter than usual, you feel heavier than usual. Eventually, however, it eases off and the door opens once again.`
			choice
				`	"Well, onwards we go."`
					goto vault
				`	"How far down are we?"`

> 			`	The Remnant smiles, and gestures "I do not know the exact distance, but suffice to say that it is far enough that multiple direct strikes to the spaceport with nuclear ordiance would be insufficient to penetrate to this depth."`

Reviewer: ```suggestion
			`	The Remnant smiles, and gestures "I do not know the exact distance, but suffice to say that it is far enough that multiple direct strikes to the spaceport with nuclear ordnance would be insufficient to penetrate to this depth."`
```

					goto walkover
				`	Wait until she notices you.`
			`	After several minutes of watching Taely commune with the trio of ships, she turns towards you and waves you over. "Enjoying the view?" She signs as you approach, but she continues without waiting for a response. "They are nice ships,and a major step forward in our ship development. Well worth watching." She nods to herself as she runs a hand along a fin, as if checking something.`
				goto merganserintro
			label walkover
			`	Taely waves to you as you approach. "Eager to meet the latest additions to our fleet?" She pats the nose of the nearest ship. "They definitely are a beautiful addition to the Remnant patrols." She smiles to herself and runs a hand along one of the fins as if checking for something.`
			label merganserintro
			branch penguin
				has "event: remnant: penguin"
			`	"Captain <first>," she signs, "Please meet our first three Mergansers. I have been working on them for years now, and recent breakthroughs in our materials science have enabled significant progress in our ship designs." She pauses and gestures to one. Would you like to touch it?`
				goto touch
			label penguin

> 			`	"Captain <first>," she signs, "Please meet our first three Mergansers. I have been working on them for years now. In fact, the penguin that you helped me complete was an early prototype that was testing some of the techniques for these. With that and other recent breakthroughs in our materials science, significant progress in our ship designs has been made possible." She pauses and gestures to one. Would you like to touch it?`

Author: I think here I mainly used it instead of them because I had already used them earlier in the paragraph to refer to the group. But yeah, them might sound better.

> 			`	"Captain <first>," she signs, "Please meet our first three Mergansers. I have been working on them for years now. In fact, the penguin that you helped me complete was an early prototype that was testing some of the techniques for these. With that and other recent breakthroughs in our materials science, significant progress in our ship designs has been made possible." She pauses and gestures to one. Would you like to touch it?`

Author: Changed "it" to "them." It does sound better.

			`	"Stop by the cafeteria tomorrow morning, <first>. There will be more you can help with, if you are interested." With that, she gestures farewell and heads off at a brisk pace.`

mission "Remnant: Cognizance 24"
	name "Equipment to Ssil Vida"
	description "Dusk needs a load of <cargo> taken to Ssil Vida."
	cargo "Outpost Equipment" 11
	passengers 1
	source
		government "Remnant"
	stopover "Esquiline"
	destination "Ssil Vida"
	to offer
		has "Remnant: Cognizance 23: done"
	on offer
		conversation
			`When you enter the cafeteria you are greeted by Dusk and Taely signing "Good morning, care to join us?" from across the room. Collecting a tray from the dispenser, you slide into an empty seat at their table. "We were just exchanging news and discovered we can combine our next assignment," signs Taely.`

> 			`	Dusk continues. "Based on the initial data we collected while we were at Ssil Vida, there is sufficient novel data to indicate a permanent research team there. Conveniently, the station is already habitable, but we will need several shipments of equipment to setup a self-contained section."`

Author: Hmmm, yeah. Changed 'indicate' to 'deserve'

					goto rush
			`	"Suit yourself, I have work to do." He quickly sets to moving the equipment to the desired room.`
				goto gavriilend
			label sure
			`	"Thanks, many hands make light work." You quickly have all the equipment moved into the desired room.`
				goto gavriilend
			label rush
			`	"Here, help carry this and we can talk while we shift this." He hands you a lift and the two of you start moving the equipment. "Basically, we had a visitor out here. Not sure who they were, but they flew a Korath Raider with a wide mix of alien technology. Our scans said there was one Korath in charge, with a crew composed of both aliens and humans. And they came with enough firepower to make life miserable for everyone. We only had a few scouts in the area, so we stayed hidden and bargained. In the end, they gave us some 'inconsequential' data, and they got an hour to explore the station and take a few loose items. Our scans indicated that they took some stuff without much obvious value from these big halls." He gestures to the room around you with all the platforms.`
			choice
				`	"That is odd."`
					goto movingon
				`	"What did the Remnant get?"`
					goto remnantgain

> 				`	"How did the Remnant get a good bargain?"`

Author: Fixed. Thanks!

			choice
				`	"Yes."`
					accept
				`	"No."`
					decline
	npc kill
		government "Korath"
		personality target marked waiting
		system
			system "Egeria"
		fleet
			names "korath"
			cargo 3
			commodities Food Clothing Metal Plastic Medical "Heavy Metals"
			outfitters "Korath Basics" "Korath Exiles"
			variant
				"Korath Chaser" 3
		dialog "You have destroyed the ships that the raider left behind."
	on waypoint
		dialog "You have patrolled all six systems."
	on complete
		payment 250000
		conversation

> 			`Back on Ssil Vida, you check in with Gavriil and report the destruction of three Korath chasers that you located. "Three?" He signs quizzically. "That is an odd number. Raiders only carry two at most. I will report this right away. But in the meantime, come take a look at what I have set up." He leads you off towards the spaceport.`

Reviewer 2: Pun. It's odd. Yet it's an odd number.

mission "Remnant: Cognizance 26"
	name "Bring an Aeon Cell to Ssil Vida"
	description "Gavriil has asked you to go pick up an Aeon Cell from Viminal by <date>."
	deadline 14
	source "Ssil Vida"
	stopover "Viminal"
	to offer
		has "Remnant: Cognizance 25: done"
	on offer
		conversation

> 			`Gavriil strides into the room with confidence. "look at this!" He gestures at the smooth sheeting covering the walls, and the new equipment quietly humming away off to the side. A large panel with a diagram of the station stands out in the middle of it.`

Reviewer 2: ```suggestion
			`Gavriil strides into the room with confidence. "Look at this!" he gestures at the smooth sheeting covering the walls while the new equipment quietly hummed away off to the side. A large panel with a diagram of the station stands out in the middle of it.`
```

			`	"I do not know what will happen to the station when the main reactor fails completely, and we are not interested in letting this station collapse before we have had the chance to thoroughly investigate it. As such, we need to take action now to give ourselves room to work." He turns towards you. "I have contacted Viminal, and they have an Aeon Cell ready for transport. Please go pick it up and transport it back here." He smiles faintly. "Oh, and it does not matter whether you transport it installed on your ship or in your cargo, but make sure it is not your primary power source: I have the tools to uninstall it from your ship and set it up to connect to the station, but I do not have the equipment to install it back into your ship." He pauses to check a readout on the display. "Remember: It is vital that I have the Aeon Cell by <date>.`
				accept
	on stopover "Viminal"
		outfit "Aeon Cell" 1

> 		dialog "The Remnant have an Aeon cell ready when you land. They carefully move it to your ship, and several touch it reverently before they depart. You remind yourself to make sure that it is suitably stowed before taking off again."

Reviewer 2: Out of curiosity, nothing happens if you fail, right? Would be fun if you fail and the entire station just disappears.

Author: Yes, it would be amusing. It's one of the more dramatic "what happens to the story if the player fails and things go differently" moments I've got marked down for development someday. I do want to have a number of divergent points in the story to make things more interesting, but my priority is having a single story complete first.

			`	"The Embers saw you safely through." Gestures Torza with a note of appreciation, appearing from a nearby hatch. "The loss of that outpost is unfortunate, but at least most of the Remnant there survived. Thank you for your assistance." He glances at his commlink. Your logistics adjustment has been added to your account, of course, but I suspect that you are interested in pushing forward the progress in studying Ssil Vida. Meet me in the spaceport when you are ready."`

mission "Remnant: Cognizance 29"
	name "Transport team to Ssil Vida."
	description "Torza has tasked you with transporting a team of technicians to Ssil Vida."
	blocked `You check in with Torza, who glances at his commlink. "You will need a ship with space for 15 people and 30 cubic meters of cargo."`
	cargo "equipment" 30
	passengers 15
	source "Caelian"
	destination "Ssil Vida"
	to offer
		has "Remnant: Cognizance 28: done"
	on offer
		conversation

> 			`You walk into the cafeteria to find a group of Remnant looking tired but pleased with themselves. Torza is striding among them with a tray filled with bottles, handing them out. "Good work, everyone. And excellent pacing on that run." He turns towards you. "And right on schedule, the captain who will be transporting you over to Ssil Vida." He gestures for you to take a seat and offers you a bottle as well.`

Reviewer 2: ```suggestion
			`You walk into the cafeteria to find a group of Remnant looking tired but pleased with themselves. Torza is striding among them with a tray filled with bottles, handing them out. "Good work, everyone. Excellent pacing on that run." He turns towards you. "And right on schedule, the captain who will be transporting you over to Ssil Vida." He gestures for you to take a seat and offers you a bottle as well.`
```

			choice
				`	Gesture that you are not thirsty.`
					goto notthirsty
				`	Take a bottle and gesture appreciation.`
			`	You sip from the bottle, and discover that it tastes like some kind of restorative sports drink. You can pick out hints of salt and a few other flavors, but most of it is an indeterminate taste you cannot quite identify. Torza nods in approval as you drink. "Especially in heat like this, hydration is important." He continues weaving around the group handing out the remaining bottles.`
			choice
				`	(drink in silence)`
					goto powerbriefing
				`	(ask the Remnant next to you what just happened)`

> 			`	The Remnant smiles a tiredly. "We did a ten kilometer run in just under thirty seven minutes, wearing our emergency environmental suits and carrying basic survival and maintenance gear. Torza set a hard pace, but we kept up." She takes a sip from her drink. "It is hard work, but endurance is rather important for survival, especially in an unknown environment like Ssil Vida." She takes another sip. "It was an interesting changes of pace. Normally I carry full engineering kit, so this was a 'fast scramble' in comparison. He," she gestures at Torza, "likes to change things up. No two runs are the same when he sets the pace."`

Reviewer 2: 3.7 minutes per klick? How buff are they?

Author: They perform very well.

I used the times from https://runninglevel.com/running-times/10k-times as a basis point, looking at the elite level for 35 year olds. I deliberately picked a time that was exceptional but not unachievable for humans in general. (note the current world-record for that distance and age bracket is 26 min and 30 mins for men and women respectively).

It is intended to be a sort of "Wow, this group of researchers is really fit! Wait a minute, they've talked about having excellent medical care, and there's those symbiotes... Is fitness really what's going on here, or is there something else giving them an edge?"

It should be clear that it *is* an achievement. They are proud of their time, and it was hard work. But it is intentionally suppose to catch the attentive person's attention that it is somewhat incongruous to have an entire technical team who just happens to be elite level runners...

mission "Remnant: Cognizance 30"
	name "Deliver datacrystal to Viminal"
	description "Gavriil has asked you to take a datacrystal to Viminal."
	landing
	source "Ssil Vida"
	destination "Viminal"
	to offer
		has "Remnant: Cognizance 29: done"
	on complete
		conversation
			`Not knowing who to deliver the data crystal to, you take it to the information desk. The Remnant currently working there gestures that they have been expecting it. They thank you for delivering it. As you leave the information desk, your commlink chimes with a message from prefect Chilia. It informs you that Chilia would like to speak with you, and is currently in the spaceport cafeteria.`

mission "Remnant: Cognizance 31"
	name "Pickup Refined Fuel"
	description "Chilia has asked you to pick up a refined fuel from Clink by <date>."
	deadline 19
	cargo "Reserved" 58
	stealth
	infiltrating
	source "Viminal"
	destination "Clink"
	to offer
		has "Remnant: Cognizance 30: done"
		has "license: Remnant Capital"
	on offer
		conversation

> 			`Chilia is just collecting his tray from the counter when you step into the cafeteria. Seeing you enter, he picks up a second one and nods towards a pair of open seats at a table with Taely and Dusk. "We are glad you could make it, <first>." Taely looks happy, brimming with contained energy. "It is an exciting day, Captain, but Dusk should lead." She nods to Dusk, who sets down his fork to gesture. "While Gavriil has been making excellent progress setting up facilities on Ssil Vida, I hae been working with a team of xenolinguists on deciphering the data we have collected so far. Based on Gavriil's recommendations, we have been focusing on the power distribution system.`

Reviewer 2: ```suggestion
			`Chilia is just collecting his tray from the counter when you step into the cafeteria. Seeing you enter, he picks up a second and nods towards a pair of open seats at a table with Taely and Dusk. "We are glad you could make it, <first>." Taely looks happy, brimming with contained energy. "It is an exciting day, Captain, but Dusk should lead." She nods to Dusk, who sets down his fork to gesture. "While Gavriil has been making excellent progress setting up facilities on Ssil Vida, I hae been working with a team of xenolinguists on deciphering the data we have collected so far. Based on Gavriil's recommendations, we have been focusing on the power distribution system.`
```

			choice
				`	"Looks great. Let's fly!"`
					goto letsfly
				`	"What sort of ship is it?"`
			`	Taely gestures in reverence. "It is what we term a liveship." She scowls at you. "And before you start thinking about ships made of big piles of flesh and ooze, this is obviously not that." She runs her hand along the ship, obviously quite smooth and solid. "Liveships, as we are calling them, are spaceships that have a sufficient degree of self-awareness and self-control to be deemed alive. They are manufactured, and have energy and fuel requirements just like any ship. Likewise, they are largely made of materials similar to what you are familiar with. But they are conscious. They can move and flex in ways that would be impossible to precisely control with servos and traditional position reporting. They can feel where each tendril is as surely as most people can tell where their hand is, and each ripple of a wormhole vortex is as perceptible to them as the crest of a sand dune would be to you or I."`

> 			`	She turns to you and pulls up a display, listing off various statistics. "From a technical standpoint, the Peregrine class starship is classified as a medium warship, built along the lines of the fast-attack cruisers from earlier eras. While it does not have the thickest armor or shields we could have installed, it makes up for it in unrivaled mobility and a neural system that has been hardened against some of the effects that could slow it down. It is also, so far, the largest production ship that has built-in cloaking." As if on queue, the ship vanishes from sight. There is a brief rustle and a whoosh of air, and the ship reappears on the other side of you. Several of the Remnant working on it look amused, and relocate to continue working. Taely gestures amusement, and continues. "Given your recent work with Plume, it may also be relevant to you that the Peregrine is what we term a 'gaslined' starship; meaning it is built to not only fly in atmosphere, but handle both the pressure and conditions found in the depths of gas giants and other high-pressure worlds.`

Reviewer 2: ```suggestion
			`	She turns to you and pulls up a display, listing off various statistics. "From a technical standpoint, the Peregrine class starship is classified as a medium warship, built along the lines of the fast-attack cruisers from earlier eras. While it does not have the thickest armor or shields we could have installed, it makes up for it in unrivalled mobility and a neural system that has been hardened against some of the effects that could slow it down. It is also, so far, the largest production ship that has built-in cloaking." As if on cue, the ship vanishes from sight. There is a brief rustle and a whoosh of air, and the ship reappears on the other side of you. Several of the Remnant working on it looks amused, and relocate to continue working. Taely gestures amusement and continues. "Given your recent work with Plume, it may also be relevant to you that the Peregrine is what we term a 'gaslined' starship; meaning it is built to not only fly in atmosphere but handle both the pressure and conditions found in the depths of gas giants and other high-pressure worlds.`
```

			choice
				`	"Sounds good. Time to go."`
					goto departure
				`	"What about people seeing the ship?"`
			`	Chilia grins. "Fortunately for us, the diversity of human preference lends itself to our cause. While there may be only a handful of ships available for sale, spaceships are a surprisingly diverse and unregulated industry. As such, shipyards have carried on the tradition dating back to the earliest of vehicles: If someone has the wealth to pay for it, there is a shipwright willing to build it. In our time observing Ancestral space, we have seen replicas in flight of every spaceship that has ever graced popular culture or a media production. While many are barely able to manage interplanetary travel, a surprising number are in use as fully functional ships plying interstellar space. Given the similarity between our ships and those of certain productions, we are not concerned with being seen - only with being scanned."`
			label departure
			`	A hatch opens on the Peregrine, and all the workers quickly finish their tasks and stow their equipment onboard. "Ready for departure when you are, Captain!" calls one from the hatch.`
				accept
	on accept
		dialog

> 			`	As the trio move clear of the landing pad Chilia signs one last message to you. "You may want to cloak as soon as you take off. Our ships will be scanning for you too: Better to practice avoiding them before you reach Ancestral space. Embers guide you, <first>!"`

Reviewer 2: Oh wow. That's devious, Zit.

Author: Thank you.

			`	As you are completing your checks your sensors alert you to the arrival of a small fleet that seems to be maintaining orbit over this area. It looks like you might have to take steps to avoid observers right away.`
				accept
	on enter "Kornephoros"
		dialog `Your comms pick up a crackling broadcast: "Don't know who you are, but there's a bounty on unaffiliated people picking up large volumes of weapons grade materials that I intend to collect!"`
	npc evade
		government "Bounty Hunter"
		personality waiting unconstrained surveillance
		system "Kornephoros"
		fleet
			names "bounty hunter"
			variant
				"Vanguard (Missile)"
				"Splinter" 2
				"Quicksilver (Proton)" 3
	on complete
		payment 250000
		event "remnant: ssil vida activation"
		conversation

> 			`When you contact Ssil Vida to confirm where to unload your cargo you are greeted by Dust, who directs you to a docking bay closer to the main reactor. As you open the hatch, though, it is Gavriil and a group of Remnant with lifters waiting. They greet you cheerfully, but they definitely seem in a rush to get the processed fuel unloaded and shuttled into the depths of the station. Once the last unit is off your ship Gavriil slows down enough to thank you for the prompt delivery, but apologizes as he rushes off as there is not much time left. His parting gesture suggests you talk to Dusk up in the control room.`

Reviewer: ```suggestion
			`When you contact Ssil Vida to confirm where to unload your cargo you are greeted by Dusk, who directs you to a docking bay closer to the main reactor. As you open the hatch, though, it is Gavriil and a group of Remnant with lifters waiting. They greet you cheerfully, but they definitely seem in a rush to get the processed fuel unloaded and shuttled into the depths of the station. Once the last unit is off your ship Gavriil slows down enough to thank you for the prompt delivery, but apologizes as he rushes off as there is not much time left. His parting gesture suggests you talk to Dusk up in the control room.`
```

			`	The final image is of the creature the Remnant termed the 'Ember Node,' floating in space. The image splinters the same way as the image of your ship, and fragments into multiple ghosts of itself that vanish into a rift of energy. The sense of curiosity returns, and then there is a another click, and suddenly your mind is your own once again.`

mission "Remnant: Cognizance 34"
	name "Postverta system sweep."
	description "Conduct a sweep of nearby systems and return to Ssil Vida."
	landing
	source "Ssil Vida"
	waypoint "Diespiter"
	waypoint "Statina"
	waypoint "Vaticanus"
	waypoint "Egeria"
	waypoint "Prosa"
	to offer
		has "Remnant: Cognizance 33: done"
	on offer
		conversation

> 			`The Remnant have a shocked expression, and a couple of them are look dazed. Most, however, seem to be handling the incursion fairly well. Dusk gestures eagerly to you. "Good to see you recovered quickly, <first>. We will take care of everyone here - we need you out there. Go do a sweep through each of the surrounding systems, and see if you can get a signal to any of the Remnant worlds. Our communicators have all lost connection, as have the isolated backups. So we need to re-establish communication promptly." He shakes his head. "I love meeting new beings, but they always manage to surprise."`

Reviewer 2: ```suggestion
			`The Remnant have a shocked expression, and a couple of them are looking dazed. Most, however, seem to be handling the incursion fairly well. Dusk gestures eagerly to you. "Good to see you recovered quickly, <first>. We will take care of everyone here - we need you out there. Go do a sweep through each of the surrounding systems, and see if you can get a signal to any of the Remnant worlds. Our communicators have all lost connection, as have the isolated backups. So we need to re-establish communication promptly." He shakes his head. "I love meeting new beings, but they always manage to surprise."`
```

			choice
				`	"What happened to us? Did we all see the same thing?"`
				`	"Where did those feelings and images come from?"`
			`	Gavriil frowns in thought. "Based on our initial survey, each person had a slightly different experience. We think that everyone had basically the same event, but as it was a series of impressions, images, and feelings, each person picked up on different aspects of it." He gestures take on a confident tone. "Which makes sense, since our life experiences shape our perceptions. Still, as to what it was..." He pauses, gathering his thoughts. "Our best guess is that this was first contact with some sort of entity." He grins at that. "I know, stating the obvious. But anyway, Dusk thinks that it might be some sort of solitary entity, as it did not use words or meanings, despite having access to our minds. As such, it uses internals to convey meaning instead of words." He glances at a map of the system. "There is, however, no indication as to where it came from.`
			choice
				`	"How do we get back?"`
					goto cutpower
				`	"What happened to the long range communication?"`

> 			`	"Well, that is a good question. We have and are running diagnostics on them, and so far they are passing all their checks. The system safeties, isolation mechanisms, security monitoring, tamper proofing, self-destruct systems, backup hardening..." He shakes his head, stymied. "Everything looks in perfect working order. The only problem is that the quantum-pairs at the heart just do not seem to have a pair anymore. In terms of quantum mechanics, it seems that the rest of the universe quite simply does not exist."`

Reviewer 2: ```suggestion
			`	"Well, that is a good question. We have, and still are, running diagnostics on them, and so far they are passing all their checks. The system safeties, isolation mechanisms, security monitoring, tamper-proofing, self-destruct systems, backup hardening..." He shakes his head, stymied. "Everything looks in perfect working order. The only problem is that the quantum pairs at the heart just do not seem to have a pair anymore. In terms of quantum mechanics, it seems that the rest of the universe quite simply does not exist."`
```

			choice
				`	"That seems problematic. How do we get back?"`
					goto cutpower
				`	"Did we destroy the universe by powering up this station?"`

> 			`	Gavrill smiles with amusement. "It is certainly a possibility. However, the fact that it was, as best we can tell, powered up previously without destroying the universe as far as we can tell suggests that it transported us rather than destroying everything else." He frowns at the communication panel. "Although everything I know about quantum mechanics refuses to countenance something being able to leave the universe so absolutely like this." He smiles again. "But that is the way of science! Finding new things in the universe that tell us that what we know is not so absolute."`

Reviewer 2: ```suggestion
			`	Gavrill smiles with amusement. "It is certainly a possibility. However, the fact that it was, as best we can tell, powered up previously without destroying the universe as far as we can tell suggests that it transported us rather than destroy everything else." He frowns at the communication panel. "Although everything I know about quantum mechanics refuses to countenance something being able to leave the universe so absolutely like this." He smiles again. "But that is the way of science! Finding new things in the universe that tells us that what we know is not so absolute."`
```

mission "Remnant: Cognizance 22"
	name "Deliver Data Backups for Taely"
	description "Deliver a data backup to <stopovers> for Taely."
	source
		government "Remnant"
	stopover
		government "Remnant"
		neighbor distance 1 3
	to offer
		has "Remnant: Cognizance 21: done"
	to complete
	on offer
		conversation
			`You step out of the <ship> into the fresh air of <origin>. You recall seeing some unusual activity over at one of the research hangars. There must be something interesting going on to generate that much activity.`
			choice
				`	Not today.`
					decline
				`	Go check out the activity.`

> 			`	You follow the stream of engineers and technicians to find Taely wedged into a cavity on what must be a ship. You cannot quite tell what it looks like, however, because it is still surrounded by scaffolding, tubes, and other systems. Whatever it is, it is quite small: Without all the construction rigging you suspect that it would probably be only slightly larger than a puffin.`

Reviewer: ```suggestion
			`	You follow the stream of engineers and technicians to find Taely wedged into a cavity on what must be a ship. You cannot quite tell what it looks like, however, because it is still surrounded by scaffolding, tubes, and other systems. Whatever it is, it is quite small: Without all the construction rigging you suspect that it would probably be only slightly larger than a Puffin.`
```

				accept
			label maybelater
			`	"Oh, that is unfortunate. Maybe some other time, then." The crowd makes various gestures of disappointment, and turn back to observing the projection and discussing their theories.`
				defer
	on stopover
		conversation
			`You land on pad D43A and sure enough, there is a team waiting to load a bunch of equipment into the racks. Within minutes they are done loading, and cheerfully wave as they head off.`
	on visit
		conversation
			`According to your log you are where you need to be, but something is missing. The mission checklist blinks quietly at you on the side of your console, reminding you that you need to have a Research Laboratory installed, and the requested equipment has to have arrived. If you have more than one ship it might not have arrived yet.`
	on complete
		require "Research Laboratory"
		conversation
			`One of the Remnant is waiting for you. "Thanks for picking up the equipment. The team that you will be taking to Ssil Vida is waiting for you in the spaceport."`

mission "Remnant: Cognizance 21"
	name "Expedition to Ssil Vida"

> 	description "Outfit a research laboratory and go to <stopover>."

Reviewer: ```suggestion
	description "Outfit a research laboratory and go to <stopovers>."
```

			choice
				`	"Yes, I can"`
					goto affirmative
				`	"I am too busy for that."`
			`	Taely looks disappointed, but shrugs. "No problem, I will get someone else to deliver it."`
				decline
			label affirmative
			`	Taely looks please. "Excellent. Just delivers this to the archive on <destination>." She hands you a very robust looking datacrystal. "Looks a bit different from other data crystals, I am sure." She pauses. "These are archival crystals, and as such they have more robust data structures with many more layers of redundancy and error correction. Likewise, they have many additional layers of encryption and security to ensure that it does not fall into enemy hands. In other words, perfect for long-term storage." She pauses and smiles. "Once you have delivered this, please come back shortly to help with the initial launch of the new prototype."`
				accept
	on stopover
		conversation

> 			`Glancing around the spaceport on <stopover> you quickly spot an information desk with several busy-looking Remnant. As you approach, one of them signs a greetings to you and asks how they can be of service today. A few gestures later, and they pull out a device that they wave over the data crystal. The device makes a number of chirps before falling silent, at which the Remnant looks satisfied. "Everything looks in order, Captain <last>. Please come this way." He gestures towards an inconspicuous hallway tucked behind the information desk.`

Reviewer: This is currently broken. It can be renamed to <stopovers>, but that would cause it to be "Glancing around the spaceport on Viminal in the Arculus system...", which would be fairly awkward when you're on the planet. Having it just be a generic "Looking around the spaceport" might be for the best. Otherwise we'd need code changes, which might not be too hard, but I'm not sure what syntax we would use for "Stopovers without the system name." <stopover planets>, I guess?

Author: Personally, I would love to have a <currentplanet> that completely ignores the mission specifications about destinations, sources, stopovers, etc. and just fills in *the object the pilot is currently standing on.*

			label departure
			`	The Remnant leads you back out of the vault the same way you entered, and back to the information desk. He thanks you for your assistance and waves farewell before being drawn into a conversation with another ship captain. You gesture a polite farewell and head back to the spaceport. Taely mentioned something about launching a new prototype, and that is likely to be interesting.`
	on complete
		payment 250000
		conversation
			`You have barely cleared the security perimeter when your commlink chimes twice with a notification of a logistics credit of <payment> being deposited to your account, closely following a message to land on pad 3E5 near the research hanger Taely has been working in.`

mission "Remnant: Cognizance 23"
	name "First Flight"

> 	description "Escort the Merganser to <stopover>."

Reviewer: ```suggestion
	description "Escort the Merganser to <stopovers>."
```

			`	The <ship> settles into the current landing pattern for the main spaceport on <destination> as you flip through the exterior camera feeds. In the distance, you notice that one of the research hangars has some activity going on around it, but otherwise, the spaceport seems to be humming along as usual. Dusk stops by the cockpit as you settle the <ship> into its berth. "Thank you for helping us, <first>. It will take us some time to analyze this data, but we will likely be mounting another expedition to Ssil Vida in the near future." He pauses, then adds, "I do not think we ever mentioned it, but, as is appropriate for your assistance, a logistics adjustment of <payment> has been applied to your account to cover your costs and provide resources to prepare for future endeavors." Dusk gives you a polite nod before heading off to help the other researchers unload the equipment.`

mission "Remnant: Cognizance 22"
	name "Deliver Data Backups for Taely"
	description "Deliver a data backup to <stopovers> for Taely."

> 	source

Author: This can be triggered on any of the three major worlds, and the return mission prior to this could end up at any of the three worlds too.

The `neighbor distance 1 5` is supposed to ensure the stopover is somewhere *other* than the planet it starts on, if that's what you were getting at.

> 	source

Reviewer 3: Huh. Not sure what happened last night, but worked fine for me today. Maybe my brain glitched out for a sec. Feel free to resolve once you've seen this!

		government "Remnant"

> 		category "Light Warship"

Author: This is suppose to limit the mission to being triggered on any planet that sells light warships.
https://github.com/endless-sky/endless-sky/wiki/CreatingMissions#filters

> 		category "Light Warship"

Reviewer 3: Strange. Let me try again later today and see if it'll activate.

> 		category "Light Warship"

Reviewer 3: `Any filter that defines a ship category will not match to systems or planets.`
I think this is your problem - seems like this is meant for missions that trigger on boarding a ship, not for ones that offer on planets. If this worked for you during testing, I'd be very confused

> 		category "Light Warship"

Author: It did, at least twice... Well, I will have to remove that, then.

> 		category "Light Warship"

Author: Ok, this has been removed and replaced by a sources filter for Thrasher turrets, which are explicitly a planet filter too. Right now only the core trio that have full Outfitters have shipyards, so this works.

			choice
				`	"Interesting."`
					goto mergansermission
				`	"What 'tackling ability?'"`
			`	"In space combat, 'tackling' refers to the tactic of pinning down an enemy to make them vulnerable to the rest of the fleet. It can also refer to hindering a target's ability to move, run, or escape. You are probably familiar with the idea of using missiles or weapons with an impact force to disrupt an opponent's hyperjump. This is a prime example of tackling." She pauses, thoughtfully. "Our inhibitors are tackling weapons, but they are not sufficiently effective as far as I am concerned."`
			choice
				`	"Thanks for the explanation."`
					goto mergansermission
				`	"What is the problem with them?"`
			`	"Decisive strike capability, mostly." Her gestures take on a conciliatory tone. "They are excellent weapons for swarms of ships, or when mounted on a ship that can carry many of them. But when one only has the opportunity to land a few hits, they are insufficient to pin down their target. But that is an aside. For now, we should celebrate our new ships, and see what they can do." She pets the nearest Merganser lovingly.`
			label mergansermission

> 			`	"But enough of that." She shakes her head and assumes a business-like expression. "We would like you to lead a pair of shakedown runs. The first is a hunt for a crippled Korath ship that got stuck in the adjacent system. We will provide you with this Merganser to fly." She pats its nose fondly. "Along with a flight of four to accompany you. Of course you are free to fly your own ship if you prefer; the important part is that you all make it back intact. So, are you up for a test flight?"`

Author: Made the change manually. Thanks!

			choice
				`	(Accept.)`
					accept
				`	(Decline.)`
					decline
	on accept
		dialog
			`	Taely smiles. "Take good care of it. Fly far, <first>!"`
		give ship Merganser

> 	npc kill

Author: Yes. It can't really run too far, so I don't see any harm in letting it run as much as it can.

mission "Remnant: Cognizance 30"
	name "Transport team to <planet>"
	description "Torza has tasked you with transporting a team of technicians to <destination>."
	blocked `You check in with Torza, who glances at his commlink. "You will need a ship with space for 15 people and 30 cubic meters of cargo."`
	cargo "equipment" 30
	passengers 15
	source "Caelian"
	destination "Ssil Vida"
	to offer
		has "Remnant: Cognizance 29: done"
	on offer
		conversation
			`You walk into the cafeteria to find a group of Remnant looking tired but pleased with themselves. Torza is striding among them with a tray filled with bottles, handing them out. "Good work, everyone. Excellent pacing on that run." He turns towards you. "And right on schedule, the captain who will be transporting you over to Ssil Vida." He gestures for you to take a seat and offers you a bottle as well.`
			choice
				`	Gesture that you are not thirsty.`
					goto notthirsty

> 				`	Take a bottle and gesture appreciation.`

Reviewer 3: ```suggestion
				`	(Gesture that you are not thirsty.)`
					goto notthirsty
				`	(Take a bottle and gesture appreciation.)`
```

Author: Made the change manually. Thank you!

mission "Remnant: Cognizance 31"
	name "Deliver data crystal to <planet>"
	description "Gavriil has asked you to take a data crystal to <destination>."
	landing
	source "Ssil Vida"
	destination "Viminal"
	to offer
		has "Remnant: Cognizance 30: done"
	on complete
		conversation
			`Not knowing who to deliver the data crystal to, you take it to the information desk. The Remnant currently working there gestures that they have been expecting it and thanks you for delivering it. As you leave the information desk, your commlink chimes with a message from Prefect Chilia: he would like to speak with you, and is currently in the spaceport cafeteria.`

mission "Remnant: Cognizance 32"
	name "Pickup Refined Fuel"
	description "Chilia has asked you to pick up a refined fuel from <planet> by <date>."
	deadline 19
	cargo "Reserved" 58
	illegal -1 "Your communication panel beeps insistently with a message that a scan has been detected, and thus the mission has been terminated."
	stealth
	infiltrating
	source "Viminal"
	destination "Clink"
	to offer
		has "Remnant: Cognizance 31: done"

> 		has "license: Remnant Capital"

Reviewer 3: Could use some blocked dialogue for this mission, or just automatic granting of the capital license. I _think_  you can get here without getting the capital license (granted in Salvage 3, only need up until Salvage 2 to meet requirements according to my brief look at the files).

Author: Hmmm... It is mainly intended as a block for people who incur the wrath of the Remnant by revealing all their stuff to the Deep, and are labeled as untrusted, and are supposed to be blocked from getting the capital ship license. Although it looks like that wasn't fully implemented in Derpy's missions. Either way, I've added a blocked text for it.

Author: I am planning for there to be some options for the Player to regain that trust, but it *is* intended to be a harder route. I have that planned for before I start on the next chapter.

			`	They step back. "Well, I will return to the controls. Just give the signal when you are ready to leave. Embers light your path, Captain <first>."`

mission "Remnant: Cognizance 36"
	name "De-activate Ssil Vida Power"
	description "Accept this to signal everyone to cut the power."
	job
	repeat
	source "Ssil Vida"
	to offer
		has "Remnant: Cognizance 35: done"
		has "Remnant: Ssil Vida Active"
	on accept
		event "remnant: ssil vida deactivation"
		conversation
			`With a shudder that is felt in every fiber of your being, you glimpse the ripple that makes reality look like the surface of a pond after a single rain drop. Almost before it begins, everything is still again. You look at your sensors, and it looks like Postverta has once again returned to its prior, barren, form.`
	on enter Postverta
		fail

> mission "Remnant: Cognizance 37"

Reviewer 3: Believe I ran into #1938 here. Not much to be done about it, though. You had the same problem?

Author: Yep. The mission won't register as being failed until the next time they land, but that's alright. Nothing depends on it being failed or completed directly, that won't be satisfied soon enough the way things are handled now. Nothing will break if/when that is ever fixed, either.

	name "Activate Ssil Vida Power"
	description "Accept this to signal everyone to restore the power."
	job
	repeat
	source "Ssil Vida"
	to offer
		has "Remnant: Cognizance 35: done"
		not "Remnant: Ssil Vida Active"
	on accept
		event "remnant: ssil vida activation"
		conversation
			`With a shudder that is felt in every fiber of your being, you glimpse the ripple that makes reality look like the surface of a pond after a single rain drop. Almost before it begins, everything is still again. You look at your sensors, and it looks like Postverta is once again overflowing with life.`
	on enter Postverta
		fail

mission "Remnant: Cognizance 38"
	name "Take data to <planet>"
	description "The rest of the Remnant need to know what is happening. Take this data to them on <destination>."
	landing
	source "Ssil Vida"
	destination "Viminal"
	to offer
		has "Remnant: Cognizance 35: done"
	on offer

> 		event "remnant: cognizance salvage expansion"

Reviewer 3: Is there somewhere in the conversation that the new availability of Chasers is noted?

Author: No, but I just added a little bit:

> He looks around, and then winks at you. "In the meantime, I have unlocked access to a few more bits of rarer Korath salvage, including access to captured Chasers.

Slightly expanded the second last sentence.

			`	The end of the message is the location of a starport on <planet>, presumably where you are supposed to go.`
				accept
	on complete
		payment 100000
		event "remnant: cognizance 1 systems available"
		dialog `As the starport computer downloads the data, your computer displays a message tacked onto the log file: "Thank you for providing the additional data from your navigational logs. It will be useful to contrast against those of our scouts. A logistics adjustment of <payment> has been authorized for the <ship>."`

mission "Remnant: Cognizance 20"
	name "Preparation for expedition"
	description "Outfit a research laboratory, retrieve equipment from <stopovers>, then bring it to <destination>."
	blocked "You need 10 free cargo space as well as a research laboratory for this mission."
	cargo "supplies" 10
	source
		government "Remnant"
	stopover
		government "Remnant"
		neighbor distance 1 5
	to offer
		has "Remnant: Cognizance 19: done"

> 		has "event: remnant: lab availability"

Reviewer 3: To the best of my knowledge and research of the files, the player can get here without having acquired the lab. As such, this is another spot where the mission chain can end abruptly without the player knowing why - and `blocked` dialog won't appear because the player hasn't even met the offer criteria. I'm cool with letting it slide for the sake of merging this PR, but let's make a note to come back to this. Maybe another mission that can offer when these criteria aren't met at least explaining why the player can't continue the story?

				accept
	npc
		government "Korath"
		personality heroic plunders harvests waiting staying
		system "Parca"
		fleet
			names "korath"
			cargo 3
			commodities Food Metal Plastic "Heavy Metals"
			outfitters "Korath Basics" "Korath Exiles"
			variant
				"Korath Dredger" 1
				"Korath Dredger (Ground Assault)" 2
				"Korath Raider (Ember)" 6
				"Korath Chaser" 24

mission "Remnant: Cognizance 28"
	name "Recover equipment from Baianus."
	description "Torza has asked you to recover equipment from Baianus by <date>."
	landing
	blocked `The scene is one of organized chaos. One Remnant signs "We need a ship that can carry 280 cargo and 20 passengers."`
	deadline 12
	cargo "equipment" 280
	passengers 20
	source "Baianus"
	destination "Caelian"
	to offer
		has "Remnant: Cognizance 27: done"
	on offer
		conversation

> 			`Punching through the Korath blockade finds you racing across the aurora filled skies of Baianus. You quickly slide into the designated approach track and start to enter your IFF code as a barrage of anti-air fire erupts from the ground to lance over your shoulder. As you drop into evasive maneuvers you spot the wreckage of a chaser tumbling to the ice behind you, and a firelance beam sears the air where the <ship> had just been. Moments later you get the confirmation signal that they have accepted your ID and are tracking your position.`

Reviewer 4: I would add an option for immediately destroying the korath, but that's just an idea.

Author: It's definitely an option I will do eventually.

			`	The shield opens one final time to allow you to leave, leaping practically into the teeth of a Korath attack. The anti-air artillery continues its rhythmic barrage against the incoming ships, but it is evident that both it and the shields around the base are failing under the assault. As you head for orbit, the last image you see is of the artillery pieces sag and collapse as they melt into the ground. Your sensors record a sudden explosion from the cavern, to which the Remnant give a sad but satisfied nod. "Take us back to Caelian please, Captain."`
				launch
	npc
		government "Korath"
		personality heroic plunders harvests waiting launching
		system "Parca"
		fleet
			names "korath"
			cargo 3
			commodities Food Metal Plastic "Heavy Metals"
			outfitters "Korath Basics" "Korath Exiles"
			variant
				"Korath Dredger" 1
				"Korath Dredger (Ground Assault)" 2
				"Korath Raider (Ember)" 6
				"Korath Chaser" 24
	on complete
		payment 500000
		conversation

> 			`The brutal sun of Caelian hits you like a hammer as you open the hatch to let the evacuees out. A few of the Remnant step into the sun as if relishing the heat, while others move quickly to the shade of a nearby structure to confer. Several, however, need to be carried out to waiting vehicles: It seems that they were hurt in one of the initial raids, and while adrenaline kept them going during the evacuation, their condition deteriorated during the flight back. The Remnant carry them with reverent haste, and they the vehicles they are loaded into vanish quickly into nearby structures.`

Reviewer 4: ```suggestion
			`The brutal sun of Caelian hits you like a hammer as you open the hatch to let the evacuees out. A few of the Remnant step into the sun as if relishing the heat, while others move quickly to the shade of a nearby structure to confer. Several, however, need to be carried out to waiting vehicles: It seems that they were hurt in one of the initial raids, and while adrenaline kept them going during the evacuation, their condition deteriorated during the flight back. The Remnant carry them with reverent haste, and then the vehicles they are loaded into vanish quickly into nearby structures.`
```

mission "Remnant: Cognizance 30"
	name "Deliver datacrystal to Viminal"
	description "Gavriil has asked you to take a datacrystal to Viminal."
	landing
	source "Ssil Vida"
	destination "Viminal"
	to offer
		has "Remnant: Cognizance 29: done"
	on complete
		conversation
			`Not knowing who to deliver the data crystal to, you take it to the information desk. The Remnant currently working there gestures that they have been expecting it. They thank you for delivering it. As you leave the information desk, your commlink chimes with a message from prefect Chilia. It informs you that Chilia would like to speak with you, and is currently in the spaceport cafeteria.`

mission "Remnant: Cognizance 31"
	name "Pickup Refined Fuel"
	description "Chilia has asked you to pick up a refined fuel from Clink by <date>."
	deadline 19
	cargo "Reserved" 58
	illegal -1 "Your communication panel beeps insistently with a message that a scan has been detected, and thus the mission has been terminated."
	stealth
	infiltrating
	source "Viminal"
	destination "Clink"
	to offer
		has "Remnant: Cognizance 30: done"
		has "license: Remnant Capital"
	on offer
		conversation

> 			`Chilia is just collecting his tray from the counter when you step into the cafeteria. Seeing you enter, he picks up a second and nods towards a pair of open seats at a table with Taely and Dusk. "We are glad you could make it, <first>." Taely looks happy, brimming with contained energy. "It is an exciting day, Captain, but Dusk should lead." She nods to Dusk, who sets down his fork to gesture. "While Gavriil has been making excellent progress setting up facilities on Ssil Vida, I hae been working with a team of xenolinguists on deciphering the data we have collected so far. Based on Gavriil's recommendations, we have been focusing on the power distribution system.`

Reviewer 5: ```suggestion
			`Chilia is just collecting his tray from the counter when you step into the cafeteria. Seeing you enter, he picks up a second and nods towards a pair of open seats at a table with Taely and Dusk. "We are glad you could make it, <first>." Taely looks happy, brimming with contained energy. "It is an exciting day, Captain, but Dusk should lead." She nods to Dusk, who sets down his fork to gesture. "While Gavriil has been making excellent progress setting up facilities on Ssil Vida, I have been working with a team of xenolinguists on deciphering the data we have collected so far. Based on Gavriil's recommendations, we have been focusing on the power distribution system.`
```

Author: I'm not sure why I can't just commit your suggestion, but in any case, thanks for spotting it. I've made the correction manually based on your suggestion.

Thanks!

			label departure
			`	The Remnant leads you back out of the vault the same way you entered, and back to the information desk. He thanks you for your assistance and waves farewell before being drawn into a conversation with another ship captain. You gesture a polite farewell and head back to the spaceport. Taely mentioned something about launching a new prototype, and that is likely to be interesting.`
	on complete
		payment 250000
		conversation
			`You have barely cleared the security perimeter when your commlink chimes twice with a notification of a logistics credit of <payment> being deposited to your account, closely following a message to land on pad 3E5 near the research hanger Taely has been working in.`

mission "Remnant: Cognizance 23"
	name "First Flight"

> 	description "Escort the Merganser to <stopovers>."

Reviewer: ```suggestion
	description "Escort the Merganser to <waypoints>, and destroy the Korath ship there."
```

	landing
	source
		government "Remnant"
	waypoint "Farinus"
	to offer
		has "Remnant: Cognizance 22: done"
	on offer
		event "remnant: cognizance merganser"
		conversation

> 			`You disembark from the <ship> just in time to watch a trio of small ships taxi out of the hangar where Taely has been working. They are strange little ships, only somewhat larger than the terns and petrels you see on a regular basis. What stands out the most, however, is the fact that the three ships seem to be walking of their own accord, moving forward on flexible landing legs. Taely follows them out, and when they reach the landing pad she whistles, causing the three ships to turn around and cluster around her. You could see her smile even from where you're standing, watching her reach up to pat them on the nose. The trio sparkle with iridescent ripples as they move around, the skin flexing and twisting like some sort of organic creature.`

Reviewer: ```suggestion
			`You disembark from the <ship> just in time to watch a trio of small ships taxi out of the hangar where Taely has been working. They are strange little ships, only somewhat larger than the Terns and Petrels you see on a regular basis. What stands out the most, however, is the fact that the three ships seem to be walking of their own accord, moving forward on flexible landing legs. Taely follows them out, and when they reach the landing pad she whistles, causing the three ships to turn around and cluster around her. You could see her smile even from where you're standing, watching her reach up to pat them on the nose. The trio sparkle with iridescent ripples as they move around, the skin flexing and twisting like some sort of organic creature.`
```

					goto walkover
				`	Wait until she notices you.`
			`	After several minutes of watching Taely commune with the trio of ships, she turns towards you and waves you over. "Enjoying the view?" She signs as you approach, but she continues without waiting for a response. "They are nice ships,and a major step forward in our ship development. Well worth watching." She nods to herself as she runs a hand along a fin, as if checking something.`
				goto merganserintro
			label walkover
			`	Taely waves to you as you approach. "Eager to meet the latest additions to our fleet?" She pats the nose of the nearest ship. "They definitely are a beautiful addition to the Remnant patrols." She smiles to herself and runs a hand along one of the fins as if checking for something.`
			label merganserintro
			branch penguin
				has "event: remnant: penguin"
			`	"Captain <first>," she signs, "Please meet our first three Mergansers. I have been working on them for years now, and recent breakthroughs in our materials science have enabled significant progress in our ship designs." She pauses and gestures to one. Would you like to touch them?`
				goto touch
			label penguin

> 			`	"Captain <first>," she signs, "Please meet our first three Mergansers. I have been working on them for years now. In fact, the penguin that you helped me complete was an early prototype that was testing some of the techniques for these. With that and other recent breakthroughs in our materials science, significant progress in our ship designs has been made possible." She pauses and gestures to one. Would you like to touch them?`

Reviewer: ```suggestion
			`	"Captain <first>," she signs, "Please meet our first three Mergansers. I have been working on them for years now. In fact, the Penguin that you helped me complete was an early prototype that was testing some of the techniques for these. With that and other recent breakthroughs in our materials science, significant progress in our ship designs has been made possible." She pauses and gestures to one. Would you like to touch them?`
```

			label touch
			choice
				`	"No."`
					goto turndown
				`	"Yes."`

> 			`	Taely smiles. "Just hold out your hand, and let it sense you first. Once it bumps your hand, you can touch it safely." You follow her example, and a few moments later the nearest Merganser nudges your hand. You follow Taely's lead in running your hand along it, and you discover that - despite the slimy appearance - the ship skin is dry and smooth. The surface is clearly metallic, but you can feel it stretch and twist under your hand as the ship shifts and moves. You can feel pulsing vibrations not entirely unlike the ones you are used to on the <ship>, but with subtle fluctuations.`

Reviewer: ```suggestion
			`	Taely smiles. "Just hold out your hand, and let it sense you first. Once it bumps your hand, you can touch it safely." You follow her example, and a few moments later the nearest Merganser nudges your hand. You follow Taely's lead in running your hand along it, and you discover that - despite the slimy appearance - the ship skin is dry and smooth. The surface is clearly metallic, but you can feel it stretch and twist under your hand as the ship shifts and moves. You can feel pulsing vibrations not entirely unlike the ones you are used to on the Penguin, but with subtle fluctuations.`
```
The player might not necessarily be flying the Penguin with this mission.

Author: No, this is intentional. It is saying that the vibrations and humms are reminiscent of whatever starship the pilot is most used to flying, but different. I'd call something like "the ship the pilot spends the most time in" but we don't track that. Next best thing is comparing to whatever the current flagship is. 

It is not specifically comparing to the Penguin.

Also, at this point in the conversation there is no

			label remnantgain
			`	You finish unloading the equipment into the room before Gavriil continues. "Well, the Korath have been around for a comparatively long time. So we asked for their astronomical dataset. The data we received was raw, uncorrected, and continuous data covering multiple tens of thousands of years of stellar observations. The data even includes data that likely corresponds to the destruction of some of their worlds during their civil war." He pauses, then continues. "Of course, we need to verify the data first. So far, though, it is checking out successfully against trusted data. If it can be validated, the data will massively expand the dataset our researchers can use to model and study the universe."`
			label movingon
			`	Gavriil tucks the lifter away into a storage alcove. "Well, thank you for your help, <first>. That definitely sped up the process."`
			label gavriilend
			`	He looks around at the stacks of equipment, and turns to you. "This will keep me busy for a few days, but if you would be willing to do some patrols, it would be much appreciated. I will have details for you once I have a few things unpacked."`

mission "Remnant: Cognizance 25"

> 	name "Equipment to Ssil Vida"

Reviewer: This is a duplicate of the last mission's name, needs to be updated.

Author: Resolved now. Thanks!

	description "Gavriil has asked you to do a patrol through the nearby systems."
	landing
	source "Ssil Vida"
	waypoint "Diespiter"
	waypoint "Statina"
	waypoint "Egeria"
	waypoint "Vaticanus"
	waypoint "Prosa"
	to offer
		has "Remnant: Cognizance 24: done"
	on offer
		event "remnant: ssil vida inhabited"
		conversation

> 			`You have barely had a chance to check to finish your preflight checks on <ship> before Gavriil is tapping on your airlock. "Setting up a new environment is always an interesting challenge." Gavriil gestures. "But to the task at hand: Could you do a quick patrol through this collection of newly accessible systems? We have scouts en route with surveillance satellites to place in each of them, but I picked up the signature of a korath raider that was here recently, and I want to be sure that it left no surprises." He pauses, then makes a gesture you have come to recognize as meaning the accompanying message is intended as a summary. "So, visit each of the systems in this sector, and make sure that the raider did not leave anything behind." His gestures switch to a tone of finality. "Can you help with this?"`

Reviewer: ```suggestion
			`You have barely had a chance to check to finish your pre-flight checks on <ship> before Gavriil is tapping on your airlock. "Setting up a new environment is always an interesting challenge." Gavriil gestures. "But to the task at hand: Could you do a quick patrol through this collection of newly accessible systems? We have scouts en route with surveillance satellites to place in each of them, but I picked up the signature of a Korath Raider that was here recently, and I want to be sure that it left no surprises." He pauses, then makes a gesture you have come to recognize as meaning the accompanying message is intended as a summary. "So, visit each of the systems in this sector, and make sure that the raider did not leave anything behind." His gestures switch to a tone of finality. "Can you help with this?"`
```

			choice
				`	"Looks great. Let's fly!"`
					goto letsfly
				`	"What sort of ship is it?"`
			`	Taely gestures in reverence. "It is what we term a liveship." She scowls at you. "And before you start thinking about ships made of big piles of flesh and ooze, this is obviously not that." She runs her hand along the ship, obviously quite smooth and solid. "Liveships, as we are calling them, are spaceships that have a sufficient degree of self-awareness and self-control to be deemed alive. They are manufactured, and have energy and fuel requirements just like any ship. Likewise, they are largely made of materials similar to what you are familiar with. But they are conscious. They can move and flex in ways that would be impossible to precisely control with servos and traditional position reporting. They can feel where each tendril is as surely as most people can tell where their hand is, and each ripple of a wormhole vortex is as perceptible to them as the crest of a sand dune would be to you or I."`

> 			`	She turns to you and pulls up a display, listing off various statistics. "From a technical standpoint, the Peregrine class starship is classified as a medium warship, built along the lines of the fast-attack cruisers from earlier eras. While it does not have the thickest armor or shields we could have installed, it makes up for it in unrivalled mobility and a neural system that has been hardened against some of the effects that could slow it down. It is also, so far, the largest production ship that has built-in cloaking." As if on cue, the ship vanishes from sight. There is a brief rustle and a whoosh of air, and the ship reappears on the other side of you. Several of the Remnant working on it looks amused, and relocate to continue working. Taely gestures amusement and continues. "Given your recent work with Plume, it may also be relevant to you that the Peregrine is what we term a 'gaslined' starship; meaning it is built to not only fly in atmosphere but handle both the pressure and conditions found in the depths of gas giants and other high-pressure worlds.`

Reviewer: > There is a brief rustle and a whoosh of air, and the ship reappears on the other side of you.

While this is pretty evocative of a ship that can turn invisible being playful, the Peregrine seems like it is a _very_ big ship. Even the Penguin is probably the size of a bus, and it would cause more than a rustle and whoosh of air if it did something like this, to say nothing of a pretty large warship. I can't imagine it wouldn't nearly knock people to the ground doing that, so the scale probably needs to be adjusted. Or maybe even just cloaking of its own volition to be cheeky would be enough.

Author: Initially I agree with you, there are basically two factors that I think make it possible:
1. It is commonly accepted that we have some kind of grav-repulsion / anti-grav tech. So the Peregrine doesn't need to blast its engines or buffet its wings to get off the ground. (the amount of air that would need to move either by wings or engines would certainly blast everyone in the area off their feat)
2. Unlike, say, a frigate, the Peregrine actually has fine control over its shape and skin. And by fine control, I mean equivalent to the control a bird has over its feathers. Which is quite a bit. 
3. The Peregrine's base shape is optimized for speed and manoeuvrability.
4. It is quite possible (although never explicitly stated) that shields might be fine tuned enough to control or manipulate air. There is, for instance, likely effectively zero friction between a shield and air, so a ship moving would only disturb the air that is actually pushed out of the way. Which is still, admittedly, quite a bit

Combining the reduced force required to move with the ability to alter its shape to minimize air resistance on an instantaneous basis could, I think, result in the possibility of the events unfolding as they have here.

That being said, this scene is intended to push the bounds of what we accept as possible and demonstrate in a very personal way just how different the Peregrine is from everything else even the Remnant have made so far. This isn't Taely waxing poetic about advances or Dawn rattling off specs. This is just pure visceral experience of watching several hundred tons of ship leap around the dock. 

This reminds me of a quote from a character in Alpha Centauri:
```
Optical computers, genetic catalogs, nanorepair modules--forget all of that. It's when you see a megaton 
of steel suspended over your head by a thread the thickness of a human hair that you really find God 
in technology.    
 - Anonymous Metagenics Dockworker, MorganLink 3DVision Live Interview
 - Accompanies the Super Tensile Solids technology
```

To me, that quote is getting at the idea that while we can talk about this or that technological advance, there's a much different psychological impact between a conceptual "cool, the ship can think for itself" and the physical "I just watched several hundred tons of ship just casually jump over a bunch of people!"

Caveat on grav related tech: It is probably extremely limited, but just barely sufficient to avoid the need for massive boosters, and to allow a slower re-entry speed that permits non-aerodynamic shapes to not get snapped off. People with better knowledge than I of physics have posited that with the ability to manipulate gravity at all, ES should be a Mass Effect style galaxy with access to wildly better propulsion and all sorts of other things that we don't see in ES.

			`	Without warning, something clicks in your mind. Suddenly there is a hum of hundreds of conversations crowding your thoughts and a momentary feeling of seeing the system through thousands of eyes. You feel a wave of curiosity, and a glimpse of Ssil Vida as it might have been once - shiny and gleaming. The image ages like a fast-forward video of untold thousands of orbits as the station wears and the sky changes around it. Then the image freezes on what appears to be the present day. A feeling of appreciation replaces the sense of curiosity, followed by a blurry image of the <ship> leaving the station and heading in a dozen different directions simultaneously, as if watching it splinter into ghosts going different directions.`
			`	The final image is of the creature the Remnant termed the 'Ember Node,' floating in space. The image splinters the same way as the image of your ship, and fragments into multiple ghosts of itself that vanish into a rift of energy. The sense of curiosity returns, and then there is a another click, and suddenly your mind is your own once again.`

mission "Remnant: Cognizance 34"

> 	name "Postverta system sweep."

Reviewer: ```suggestion
	name "Postverta system sweep"
```

			`	He leads you over to a panel in the main loading dock. "I know, it says 'Job Board' on it. I re-used the coding for our task list to improvise this mass-notification system." He smiles in amusement. "When you are ready to go, just select the 'De-activate Array' option from the panel. That will notify everyone and begin the shutdown." He hands you a datacrystal, and then a large case. "Here is an encrypted datacrystal of all our data so far. The case has several sets, with the data stored in a variety of different mechanisms and encryptions. We do not know what happens when the station is activated or deactivated, so multiple formats for storing the data will give a better chance that at least some of it will remain intact to be delivered." He steps back. "Well, I will return to the controls. Just give the signal when you are ready to leave. Embers light your path, Captain <first>."`

mission "Remnant: Cognizance 35"

> 	name "De-activate Ssil Vida Power."

Reviewer: ```suggestion
	name "De-activate Ssil Vida Power"
```

	description "Accept this to signal everyone to cut the power."
	invisible
	job
	repeat
	source "Ssil Vida"
	to offer
		has "Remnant: Cognizance 34: done"
		has "Remnant: Ssil Vida Active"
	on accept
		event "remnant: ssil vida deactivation"
		conversation
			`With a shudder that is felt in every fiber of your being, you glimpse the ripple that makes reality look like the surface of a pond after a single rain drop. Almost before it begins, everything is still again. You look at your sensors, and it looks like Postverta has once again returned to its prior, barren, form.`
			action
				fail

mission "Remnant: Cognizance 36"

> 	name "Activate Ssil Vida Power."

Reviewer: ```suggestion
	name "Activate Ssil Vida Power"
```

	description "Accept this to signal everyone to restore the power."

> 	invisible

Reviewer: ```suggestion
```
Can't use the job if it's invisible!
```

Author: Doesn't `invisible` just control if it shows up in the active jobs list (and whether it shows the destination marker)?

I was hoping it would still be there in the job board.

Reviewer: It takes up a spot on the job board, but it empty and can't be selected by the player. I think you'll just need to have it hang around in the player's log for a bit, or otherwise would have to do it through the spaceport or the like.

			action
				set "Remnant: Cognizance Neural Net Explanation"
			`	A third Remnant steps up. "As to the matter at hand, this means that the sooner any of us hear your report, the sooner everyone will have heard it." They look at you expectantly.`
				goto cognizanceexplanation
			label cognizanceuntrusted
			`	The Remnant smiles mysteriously. "A society has to have some secrets, does it not? Let us just say that our communication networks are very effective. At this point, just know that the sooner you give the report, the sooner that everyone will be able to work on this information." They pause, then continue. "And that does, of course, includes the prefects."`

> 			label cognizanceexplanation
> 			choice
> 				`	"OK, I will explain what happened."`
> 				`	"Alright, you convinced me."`

Reviewer: ```suggestion
			choice
				`	"OK, I will explain what happened."`
				`	"Alright, you convinced me."`
			label cognizanceexplanation
```
Jumping to the explanation will cause those two choices to pop up, so the label should be on the other side.

			choice
				`	"Thanks! That was more detail than I was expecting."`
				`	"Okay, that satisfied my curiosity for now."`
			label acceptance
			`	"Excellent. You deal with the Korath, and I will file a report about this threat. Please inform me immediately once you have taken care of it." He pauses, then continues, "Oh, and remember to record the battle. It may be useful in understanding what they are doing there."`
				accept
	on enter Nenia
		dialog
			`As you spot the Korath hunters you recall Plume's request and tap the 'Record' button as your ship prepares for combat.`
	npc kill
		government "Korath"
		system "Nenia"
		personality forbearing timid uninterested surveillance staying
		fleet
			names "korath"
			cargo 1
			commodities "void sprite parts"
			variant
				"Korath Raider (Ember)"
				"Korath Chaser" 2
		dialog "You have destroyed the Korath hunters. You can now return to <planet> to collect your payment."
	on complete
		event "korath hunting"

> 		conversation

Author: It's basically just a pointer to prevent the Player from getting lost. It has no content in and of itself.

			label next
			`	 "It is imperative that all the Korath involved in this exploration be located immediately. According to our surveillance, that ship was part of a fleet that recently passed through Parca. Locate the fleet and report back."`
				accept
	npc kill
		to despawn
			has "remnant: cognizance: found korath fleet"
		government "Korath"
		personality staying surveillance
		system "Aescolanus"
		fleet
			names "korath"
			cargo 2
			commodities "void sprite parts"
			variant
				"Korath Dredger"
				"Korath Raider (Ember)" 4
				"Korath Chaser" 12
		conversation
			apply
				set "remnant: cognizance: killed korath fleet"
			`You've destroyed the Korath fleet that was harassing the void sprites.`
	on enter "Aescolanus"
		set "remnant: cognizance: found korath fleet"

> 		dialog `You have located the Korath fleet. They do not seem to be going anywhere at the moment.`

Author: I know this is not specifically a Remnant speaking here, but I prefer to keep the entirety of the Remnant missions fully written out. Let's just say it is intended to help deepen the immersion for the player of being in a culture without contractions to the point they even don't use them while thinking. At least, while in Remnant missions, anyway.

> 		dialog `You have located the Korath fleet. They do not seem to be going anywhere at the moment.`

Reviewer 3: Fair enough!

mission "Remnant: Cognizance 9"
	name "Transport engineer to <planet>"
	description "An engineer working for Plume needs transportation for himself and some equipment to <planet>."
	blocked "Plume's engineer needs a free bunk and <tons> of cargo space for his equipment. Come back when you have enough space."
	cargo equipment 8
	passengers 1
	source "Aventine"
	destination "Esquiline"
	to offer
		has "Remnant: Cognizance 8: done"
	on offer
		log "People" "Gavriil" `A technician that works with Plume, and a talented astrobiologist. Gavriil is the project lead for the repurposed Esquiline station.`
		conversation
			`You walk into Plume's lab and are greeted by a large stack of crates and equipment. Plume and another Remnant are gathered around the stack checking items off a list. "Ah, <first>! Good timing. We were just finishing the inventory of supplies that need to go on this delivery. He gestures to the engineer. "This is Gavriil, one of our best technologists and a skilled astrobiologist in his own right. He will be going with you."`

> 			`	"It will be a pleasure to work with you," signs Gavriil, making a short bow. "We will be traveling to a research station on <planet>. Yes, it is a gas giant." He smiles at that. "I have to get some equipment setup and get the core systems online. There will be a few more loads of materials afterwards." He looks askance at you. "Shall we get started?"`

Author: Somewhat? I was going for more of an "eager to get going" sort of vibe. Not quite plaintive, but just a hint of it.

			branch penguin
				has "event: remnant: penguin"
			`	She turns to face the group of captains and gestures with tones of respect. "May the Embers bring interest to your journey."`
				accept
			label penguin
			`	She nods at you and continues. "The Penguin will be even better than the Puffin for this task, and you are welcome to use her, but we do not have any available to accompany you. Continuing to rely on Puffins is not ideal, but we will make do with what we have." She turns to face the group of captains and gestures with tones of respect. "May the Embers bring interest to your journey."`
				accept
	on stopover
		dialog
			`You cruise through the atmosphere at a relaxed pace, waiting while the equipment does its work. You see fewer void sprites than usual, and many have scars. Once the equipment indicates that it has reached capacity, you climb up to where the Pelican is waiting in the high atmosphere, hook up the connections to offload the gas, and then head back down into the depths of the planet. After several hours, the captain of the Pelican informs you that the tanks are full, and it is time to head to <planet>.`
	on complete
		event "remnant: postverta reveal"

> 		conversation

Author: If the block of conversation felt like it was one contiguous set, I kept it as one conversation so that the player didn't have back to back conversation boxes popping up. I personally find that a bit disruptive. It's OK if it is unrelated missions, but annoying when it is two parts of the same mission.

	

mission "Remnant: Cognizance 17"
	landing
	name "Report back to <planet>"
	description "The Remnant will want to know about what happened on Nasqueron."
	source "Nasqueron"
	destination "Viminal"
	to offer
		has "Remnant: Cognizance 16: active"
	on offer
		fail "Remnant: Cognizance 16"
		event "remnant: nenia ember waste node"
		conversation
			`You follow the creature down through the clouds of Nasqueron and emerge into what can only be described as an amphitheater in the clouds. Thousands of void sprites fill the area, and the creature you have been following floats near the center of it. Your external sensors pick up myriad pulses of energy and eerie whistling noises that seem to resonate in the clouds. As the <ship> moves forward, there is a sudden silence, and the flickers of energy synchronize.`
			`	You feel a sudden pressure in your mind and a sense of presence as the world around you fades out somewhat.`

> 			`	"Know you not, do I. Live creature inside not-quite creature, like ones who flew but not everywhere." You are momentarily disoriented as you look around and see hundreds of views of yourself peering out the cockpit window. "Your intention reverberates - Know and not know - Return with more echoes - Perhaps I shall hear."`

Author: Thank you!

			`	Plume signals his cautious agreement. "This message is intriguing, but I agree it should not be our first priority. That being said, I will plan for additional investigations after we have completed this initial operation."`
			`	The two look at you, and Chilia authoritatively signs, "We are evacuating a large group of void sprites from Nenia to Esquiline. The atmospheres are largely a match, and we have reconfigured the station to provide the nutrients that aren't otherwise present. We have a fleet of Pelicans that have basically been hollowed out so we can lure as many void sprites into it as we can."`
			choice
				`	"How are we supposed to lure them inside?"`
				`	"Are we sure this is a good idea?"`
					goto goodidea
				`	"Okay, let's go do this."`
					goto letsgo

> 			`	Plume looks a bit embarrassed. "Centuries ago when we first studied the void sprites, we learned how to harvest them. As part of that, we discovered that they were particularly interested in certain types of radiation. Likewise, if they discovered a certain kind of element common in some asteroids, they would ignore almost everything while they chewed on it. So, we have emitters inside the cargo holds along with asteroids to keep them busy for a short while."`

Author: Should that be "When" as opposed to "Then"?

> 			`	Plume looks a bit embarrassed. "Centuries ago when we first studied the void sprites, we learned how to harvest them. As part of that, we discovered that they were particularly interested in certain types of radiation. Likewise, if they discovered a certain kind of element common in some asteroids, they would ignore almost everything while they chewed on it. So, we have emitters inside the cargo holds along with asteroids to keep them busy for a short while."`

Author: Fixed manually. Thanks!

				defer
	on stopover
		conversation
			`You land on pad D43A and sure enough, there is a team waiting to load a bunch of equipment into the racks. Within minutes they are done loading, and cheerfully wave as they head off.`
	on visit
		conversation
			`According to your log you are where you need to be, but something is missing. The mission checklist blinks quietly at you on the side of your console, reminding you that you need to have a Research Laboratory installed, and the requested equipment has to have arrived. If you have more than one ship it might not have arrived yet.`
	on complete
		require "Research Laboratory"
		conversation
			`One of the Remnant is waiting for you. "Thanks for picking up the equipment. The team that you will be taking to Ssil Vida is waiting for you in the spaceport."`

mission "Remnant: Cognizance 21"
	name "Expedition to Ssil Vida"
	description "Outfit a research laboratory and go to <stopovers>."
	blocked "You need 10 free space for this mission."
	cargo "supplies" 10
	source
		government "Remnant"
	stopover
		planet "Ssil Vida"
	to offer
		has "Remnant: Cognizance 20: done"
	on offer
		require "Research Laboratory"
		conversation

> 			`	In the cafeteria the scene hasn't changed much: There is still a large group of Remnant gathered around a cluster of projectors, with a constant stream of gestures discussing various points of the data. You understand enough to figure out that most of the conversation is deep into technical discussions of how the lifesign sensors might give the readings they do without having anything visible. The principle differences from before is that there is a stack of small bags and helmets next to the door and five of the Remnant present are wearing pressure suits. One of them notices you and waves you over. "Captain <first>! My name is Dusk, and I am one of the researchers that you volunteered to transport to Ssil Vida."`

Author: Yes, it should.

			choice
				`	"Dusk? Are names of like that common?"`
				`	"Pleased to meet you, Dusk."`
					goto missiondescription
			`	"You mean names of the various phases of daylight?" He pauses for a few seconds. "I would not say they are common, but nor are they rare, either. Our ancestors devoted their lives, one way or another, to studying the stars and the ever-changing light of the universe. It should not be surprising that their vocabulary should be so entwined in our culture."`
			choice
				`	"Who were your ancestors?"`
				`	"Interesting. Well, what is our mission?"`
					goto missiondescription

> 			`	Dusk looks thoughtful for a moment. "Our ancestors came from a different era for humanity. They were the heirs of a golden age of human scientific progress and progress, when governments and institutions poured vast wealth into science. The developments of that time period resulted in the completion of the hyperspace relay network, new terraforming techniques, and advanced genetic engineering capabilities." His face clouds at that. "This, of course, culminated in the Alphas." He pauses, then shakes his head. "But you asked who are ancestors were, not about human history.`

Author: Oh, it should be "human scientific development and progress"

> 			`	Dusk looks thoughtful for a moment. "Our ancestors came from a different era for humanity. They were the heirs of a golden age of human scientific progress and progress, when governments and institutions poured vast wealth into science. The developments of that time period resulted in the completion of the hyperspace relay network, new terraforming techniques, and advanced genetic engineering capabilities." His face clouds at that. "This, of course, culminated in the Alphas." He pauses, then shakes his head. "But you asked who are ancestors were, not about human history.`

Reviewer 3: ```suggestion
			`	Dusk looks thoughtful for a moment. "Our ancestors came from a different era for humanity. They were the heirs of a golden age of human scientific development and progress, when governments and institutions poured vast wealth into science. The developments of that time period resulted in the completion of the hyperspace relay network, new terraforming techniques, and advanced genetic engineering capabilities." His face clouds at that. "This, of course, culminated in the Alphas." He pauses, then shakes his head. "But you asked who are ancestors were, not about human history.`
```
There ya go!

> 			`	"In that era there were a number of burgeoning astronomy and exploration programs across human space. Despite also benefiting from the substantial budgets science in general as receiving, they were being pushed to the margins of the academic community: Unlike many of the more practical sciences like biology and chemistry which were coming up with new wonders, astronomy and astrophysics was purely academic. There was no budget in explaining the whys of the universe, not when other scientists were creating homo superior and unleashing instantaneous interstellar communication.`

Author: It is saying that while the overall field of science was getting bigger budgets, astronomy and astrophysics got shoved to the backburner with small funding increases. If every department in the university gets a seven figure budget increase except for astrophysics, which only gets a six figure increase, guess who gets shoved to the worst part of campus? They weren't *losing* budget, but they didn't get nearly as much of the largess as the other fields of study.

> 			`	"In that era there were a number of burgeoning astronomy and exploration programs across human space. Despite also benefiting from the substantial budgets science in general as receiving, they were being pushed to the margins of the academic community: Unlike many of the more practical sciences like biology and chemistry which were coming up with new wonders, astronomy and astrophysics was purely academic. There was no budget in explaining the whys of the universe, not when other scientists were creating homo superior and unleashing instantaneous interstellar communication.`

Reviewer 3: Ahh, gotcha. Looks like this typo caused my confusion. I added in just a little to help with clarity too.
```suggestion
			`	"In that era there were a number of burgeoning astronomy and exploration programs across human space. Despite also benefiting from the substantial budget increases that science in general was receiving, they were being pushed to the margins of the academic community: Unlike many of the more practical sciences like biology and chemistry which were coming up with new wonders, astronomy and astrophysics was purely academic. There was no budget in explaining the whys of the universe, not when other scientists were creating homo superior and unleashing instantaneous interstellar communication.`
```

			choice
				`	"They were worried about the course the war was taking?"`
				`	"That would be worrisome. We should focus on the mission."`
					goto missiontransition
			`	Dusk starts gesturing again. "Yes, but also by what they saw. Strange ships that slipped over the horizon, odd energy signatures in the distance, novel equipment that simply showed up on Deep ships without explanation... It seemed like someone out there was taking advantage of the chaos to influence humanity. Someone beyond the reach of the normal hyperspace network. It spurred our ancestors to revisit old theories and kindle new ones. Knowing that there was someone, something beyond what we knew made us bold. It was not long before the first ship traveled through the wormhole."`
			choice
				`	"Which ship was that?"`
				`	"I think I can see where this is going. Time to get to work."`
					goto missiontransition

> 			`	Dusk smiles "The Shadow Step, an independent science vessel who did double-duty as a gas miner. She made a quiet fortune harvesting rarer gases from obscure gas planets off the charted paths. Never enough to attract the attention of the big corporate entities, but enough to be comfortable. Her captain was always making modifications. Not a genius by any means, but what they lacked in brilliance, they made up for in dedication and willingness to experiment. According to our records, making modifications to the Shadow Step was their main hobby, and the small crew she flew with had similar interests." Dusk pauses to take a drink.`

Author: Yes, but changing it to be neutral/ambiguous is fine too.

mission "Remnant: Cognizance 22"
	name "Deliver Data Backups for Taely"
	description "Deliver a data backup to <stopovers> for Taely."
	source
		government "Remnant"
	stopover
		government "Remnant"
		neighbor distance 1 3
	to offer
		has "Remnant: Cognizance 21: done"
	to complete
	on offer
		conversation
			`You step out of the <ship> into the fresh air of <origin>. You recall seeing some unusual activity over at one of the research hangars. There must be something interesting going on to generate that much activity.`
			choice
				`	Not today.`
					decline
				`	Go check out the activity.`
			`	You follow the stream of engineers and technicians to find Taely wedged into a cavity on what must be a ship. You cannot quite tell what it looks like, however, because it is still surrounded by scaffolding, tubes, and other systems. Whatever it is, it is quite small: Without all the construction rigging you suspect that it would probably be only slightly larger than a Puffin.`

> 			`	As you peer around the supports a Remnant scurries up with a slimy cable in a bag and hands it to you. "Here, Taely will need this in a few seconds, could you hand it to her when she asks for it? I need to go find the next part. Thanks!" The Remnant vanishes into an adjacent storage room before you even have the chance to respond. Almost immediately, however, Taely twists around to look back at you. "Oh, <first>! I was expecting Cora to be there with the... Ah, they gave it to you. Excellent. If you could just hand it up to me, it would be much appreciated." You reach up to hand her the bag, and Taely inches farther into the craft until only her feet are visible. A few liquid sounds later, and Taely slides back out and drops to the ground.`

Author: Hehe. Yeah.

			`	She pats a piece of exposed hull, and you think you can almost feel some kind of vibration or hum in response. Taely takes on a distant expression, as if lost in thought. After a long enough pause to feel awkward, she blinks a few times and focuses on you again. "Sorry about that. A new ship waking up and all that." She smiles, thoughtfully. "Before I get absorbed in my work again, would you be able to make a delivery for me? It is a simple one, but important. I need someone to deliver a crate of data files to <destination>. As my development project has reached its end, it is time to send a hard copy of the data to one of our archives for safe keeping. Could you deliver it for me?"`
			choice
				`	"Yes, I can"`
					goto affirmative
				`	"I am too busy for that."`
			`	Taely looks disappointed, but shrugs. "No problem, I will get someone else to deliver it."`

> 				decline

Author: Agreed. For now, swapped the decline to a defer. The initial description is suitable for running into a second time.

			branch trusted
				and
					has "license: Remnant Capital"
					or
						not "Deep: Remnant 0: offered"
						has "Deep: Remnant 0: declined"
						has "Deep: Remnant 3: Secret: done"
			label surfacevault
			`	The Remnant leads you through several sets of increasingly impressive doors before arriving at a room filled with tiny doors. They pick one, use a key to open one, and pull out a small metal box that they unlock with a second key. "If you would place the data crystal here, we will record the surface metadata, and those who need it will be able to retrieve it when necessary." He waits until you have placed it in the box, then reverses the process to secure it in the vault wall again.`
				goto departure
			label trusted
			`	The Remnant gestures "Would you prefer to put it in the surface vault or the deep vault?"`

> 			choice

Author: Sure, I should do that. It's basically a "You can deposit it up here and we'll take care of it, or you can go through the long process to deliver it down below yourself." Or in other words, do you just want to get on with stuff, or do you want conversations and lore?

			choice
				`	"Well, onwards we go."`
					goto vault
				`	"How far down are we?"`
			`	The Remnant smiles, and gestures "I do not know the exact distance, but suffice to say that it is far enough that multiple direct strikes to the spaceport with nuclear ordnance would be insufficient to penetrate to this depth."`
			label vault
			`	Outside the elevator, the Remnant leads you down a short hallway with many doors, and eventually it opens onto a larger hall. At the opposite end of the hall is a massive door flanked with several large devices that look remarkably similar to Point Defense Turrets, but subtly different. Your guide takes a few steps into the room, and a short pedestal pops out of the floor near him. He pulls a license out of his pocket, places it in the slot, and presses firmly on it for a second, before removing it once again.`

> 			`	He then walks briskly across the hall, gesturing you to follow. Shields flare in and out of existence around the two of you, forming a series of openings that allow you to approach the vault door, which opens when you reach it. Inside is a disappointingly small room with a small tray on a pedestal.`

Author: Classic force fields, or projected shields.

> 			`	He then walks briskly across the hall, gesturing you to follow. Shields flare in and out of existence around the two of you, forming a series of openings that allow you to approach the vault door, which opens when you reach it. Inside is a disappointingly small room with a small tray on a pedestal.`

Reviewer 3: So the shields form `a series of openings`? That's what the current wording implies to me.

> 			`	He then walks briskly across the hall, gesturing you to follow. Shields flare in and out of existence around the two of you, forming a series of openings that allow you to approach the vault door, which opens when you reach it. Inside is a disappointingly small room with a small tray on a pedestal.`

Author: @W1zrad The intent was something akin to the series of force fields during the Obi wan vs Darth Maul fight (youtube link: https://youtu.be/Qp91xlsaKSo?t=98 )

			`	Dusk continues. "Based on the initial data we collected while we were at Ssil Vida, there is sufficient novel data to deserve a permanent research team there. Conveniently, the station is already habitable, but we will need several shipments of equipment to setup a self-contained section."`
			choice
				`	"OK, clear enough."`
					goto skipexplanation
				`	"Why do you need to do that?"`
			`	"For safety. The station might appear to be enduring and constant, but if we lacked independent facilities, the slightest failure could kill everyone. Given that we do not even understand how the station is running, depending on it moving forward would be foolish." Taely nods in agreement with Dusk, and adds. "We will, of course, be looking to do a full evaluation of the station eventually, but it is rather massive and it will be quite some time before we can finish that.`
			label skipexplanation

> 			`	Dusk gestures and their commlink sprays a holographic list into the air between you. "Basically, we would like you to load this set of equipment, then stop by Esquiline to pick up a technician by the name of Gavriil, who will be leading the establishment of the life support systems on Ssil Vida."`

Author: I was intentionally portraying Dusk as being somewhat Androgynous and/or ambiguous with referring to them as they/them. Does this work for doing that?

> 			`	Dusk gestures and their commlink sprays a holographic list into the air between you. "Basically, we would like you to load this set of equipment, then stop by Esquiline to pick up a technician by the name of Gavriil, who will be leading the establishment of the life support systems on Ssil Vida."`

Reviewer 3: Cool with me! The reason I suggested this change is because I remember Dusk already being referred to as a he/him (maybe in #6153). I'll double check/correct #6153 myself so Becca doesn't have to worry about it if you're cool with double checking this file for it.

			`	You find Gavriil nearby, where he is going over a freshly installed hydroponics array with a pair of Remnant in tow. He quickly finishes and gestures farewell to the other two before walking over to you. "Greetings, Captain <first>! Welcome back to Esquiline, it has changed a bit since you were here last." His gestures have an overtone of pride. "Dusk told me that we have a new station to bring online again, so we should probably get moving.`
	on complete
		payment 250000
		conversation
			`	The equipment is quickly unloaded as Gavriil quickly scouts some of the nearby rooms. "I have picked a suitable area to start in. If you could help move these over to that room there," He gestures towards one of the large rooms with the platforms in it. "It would be much appreciated."`
			choice
				`	"I would rather not."`
				`	"Sure."`
					goto sure
				`	"Why the rush?"`
					goto rush
			`	"Suit yourself, I have work to do." He quickly sets to moving the equipment to the desired room.`
				goto gavriilend
			label sure
			`	"Thanks, many hands make light work." You quickly have all the equipment moved into the desired room.`
				goto gavriilend
			label rush

> 			`	"Here, help carry this and we can talk while we shift this." He hands you a lift and the two of you start moving the equipment. "Basically, we had a visitor out here. Not sure who they were, but they flew a Korath Raider with a wide mix of alien technology. Our scans said there was one Korath in charge, with a crew composed of both aliens and humans. And they came with enough firepower to make life miserable for everyone. We only had a few scouts in the area, so we stayed hidden and bargained. In the end, they gave us some 'inconsequential' data, and they got an hour to explore the station and take a few loose items. Our scans indicated that they took some stuff without much obvious value from these big halls." He gestures to the room around you with all the platforms.`

Author: Should that be "help me carry" rather than "help carry me"?

> 			`	"Here, help carry this and we can talk while we shift this." He hands you a lift and the two of you start moving the equipment. "Basically, we had a visitor out here. Not sure who they were, but they flew a Korath Raider with a wide mix of alien technology. Our scans said there was one Korath in charge, with a crew composed of both aliens and humans. And they came with enough firepower to make life miserable for everyone. We only had a few scouts in the area, so we stayed hidden and bargained. In the end, they gave us some 'inconsequential' data, and they got an hour to explore the station and take a few loose items. Our scans indicated that they took some stuff without much obvious value from these big halls." He gestures to the room around you with all the platforms.`

Reviewer 3: Yes! Here's a correction:
```suggestion
			`	"Here, help me carry this and we can talk while we shift this." He hands you a lift and the two of you start moving the equipment. "Basically, we had a visitor out here. Not sure who they were, but they flew a Korath Raider with a wide mix of alien technology. Our scans said there was one Korath in charge, with a crew composed of both aliens and humans. And they came with enough firepower to make life miserable for everyone. We only had a few scouts in the area, so we stayed hidden and bargained. In the end, they gave us some 'inconsequential' data, and they got an hour to explore the station and take a few loose items. Our scans indicated that they took some stuff without much obvious value from these big halls." He gestures to the room around you with all the platforms.`
```

				accept
	on stopover "Viminal"
		outfit "Aeon Cell" 1
		dialog "The Remnant have an Aeon cell ready when you land. They carefully move it to your ship, and several touch it reverently before they depart. You remind yourself to make sure that it is suitably stowed before taking off again."
	on complete
		payment 250000
		outfit "Aeon Cell" -1

> 		conversation

Author: Done. Created Cog 27 so there is a mission with a description and a marker so the player will have a reminder of where they need to go.

	deadline 12
	cargo "equipment" 280
	passengers 20
	source "Baianus"
	destination "Caelian"
	to offer
		has "Remnant: Cognizance 27: done"
	on offer
		conversation
			`Punching through the Korath blockade finds you racing across the aurora filled skies of Baianus. You quickly slide into the designated approach track and start to enter your IFF code as a barrage of anti-air fire erupts from the ground to lance over your shoulder. As you drop into evasive maneuvers you spot the wreckage of a chaser tumbling to the ice behind you, and a firelance beam sears the air where the <ship> had just been. Moments later you get the confirmation signal that they have accepted your ID and are tracking your position.`
			`	As you approach the entrance to the starport directions appear on your comm panel directing you to a spot where a large stack of equipment surrounds the landing pad. You ease inside and have barely touched down when a technician is dropping from the ceiling down onto your hull with a refueling hose and repair equipment. Meanwhile, a swarm of Remnant have equipment in motion. You quickly open the cargo hold doors, and the Remnant rush onboard directing lifts and loaders.`

> 			`	The Remnant move fast, but you can hear the battle approaching. Even through the <ship>'s dampeners you can feel the heavy thud of anti-air artillery. The whine of generators being pushed to the limit fills the air as the shields crackle with incoming fire. Through the cacophony the Remnant move with purpose, gesturing instructions silently through the din. Finally the last load is packed onboard, and one of them comes up to the cockpit. "Thank you for your assistance, Captain <last>. As the explosive charges have been set, I recommend we leave promptly." They nod respectfully, then strap themselves into the nearest harness.`

Author: Not for the main PR, but for a follow up PR with branches and stuff... ;)

			`	Torza makes a gesture of respect. "Embers sharpen your sight, friends."`
				accept
	on complete
		payment 250000
		conversation
			`Gavriil is waiting at the dock when the <ship>'s hatches finish cycling. "Welcome," he gestures. "It is good to have fresh minds to share Ssil Vida with." The Remnant onboard stream out of the ship in a rush, each carrying a heavy pack of equipment. They gesture respect, and head off in different directions. Evidently they have studied the maps prior to arrival, as they all seem to know where they are going. Several stay behind with Gavriil and help unload the rest of the equipment.`
			`	"We have made good progress since you were here last. We have managed to take over control of life support systems in several wings of rooms, and now those are run entirely from our own systems. Aside from relieving a tiny amount of load from the station's power core, it also means that the station could fail completely and we would still have safe living quarters to serve as base of operations."`

> 			`	He pauses, and is gestures take on a tone of disgust. "Of course, we are working to avoid that. But having multiple backup systems is the key to survival in hostile conditions."`

Author: He's a professional. Letting the main reactor fail on his watch would be a personal insult. That sort of attitude, anyway.

			`	There, crouched next to <ship>, is what you can only describe as a predator: It has long sleek lines with flared wings that resemble feathers, and a massive engine array that suggest incomparable mobility in a ship of its size. As you approach you can see several Remnant working around it. One gestures, and the ship extends a wing and the Remnant reaches in between the feathers to do something, then steps back and the ship runs through a range of twisting motions as it moves its wings around. It strongly resembles a raptor preening its feathers.`
			`	Taely walks right up to the nose and the ship crouches down so she can put her hand on its nose. "Captain <first>, I would like to introduce you to the Peregrine. It is the first new capital warship design we have built in over a century, and is the result of decades of work on my part as well as many others. Hopefully, it is the first of many new ships that are successful."`
			choice
				`	"Looks great. Let's fly!"`
					goto letsfly
				`	"What sort of ship is it?"`

> 			`	Taely gestures in reverence. "It is what we term a liveship." She scowls at you. "And before you start thinking about ships made of big piles of flesh and ooze, this is obviously not that." She runs her hand along the ship, obviously quite smooth and solid. "Liveships, as we are calling them, are spaceships that have a sufficient degree of self-awareness and self-control to be deemed alive. They are manufactured, and have energy and fuel requirements just like any ship. Likewise, they are largely made of materials similar to what you are familiar with. But they are conscious. They can move and flex in ways that would be impossible to precisely control with servos and traditional position reporting. They can feel where each tendril is as surely as most people can tell where their hand is, and each ripple of a wormhole vortex is as perceptible to them as the crest of a sand dune would be to you or I."`

Reviewer 3: This raises some... interesting humanitarian thoughts. Do the ships get to _choose_ whether they're put in harm's way? Reminds me a little of using, say, horses in a medieval war. I don't think this is by any means something I'm concerned about, but I do think there's room for some interesting discussion about this. Maybe for future content? Something to consider incorporating into future Remnant stories if you weren't already.

Author: Yep, it's already being planned. Nice to see you pick up on it. :)

			choice
				`	"Sounds good. Time to go."`
					goto departure
				`	"What about people seeing the ship?"`
			`	Chilia grins. "Fortunately for us, the diversity of human preference lends itself to our cause. While there may be only a handful of ships available for sale, spaceships are a surprisingly diverse and unregulated industry. As such, shipyards have carried on the tradition dating back to the earliest of vehicles: If someone has the wealth to pay for it, there is a shipwright willing to build it.`
			`	In our time observing Ancestral space, we have seen replicas in flight of every spaceship that has ever graced popular culture or a media production. While many are barely able to manage interplanetary travel, a surprising number are in use as fully functional ships plying interstellar space. Given the similarity between our ships and those of certain productions, we are not concerned with being seen - only with being scanned."`
			label departure
			`	A hatch opens on the Peregrine, and all the workers quickly finish their tasks and stow their equipment onboard. "Ready for departure when you are, Captain!" calls one from the hatch.`
				accept
	on accept
		dialog

> 			`	As the trio move clear of the landing pad Chilia signs one last message to you. "You may want to cloak as soon as you take off. Our ships will be scanning for you too: Better to practice avoiding them before you reach Ancestral space. Embers guide you, <first>!"`

Reviewer 3: ```suggestion
			`	As the trio move clear of the landing pad Chilia signs one last message to you. "You may want to cloak as soon as you take off. Our ships will be scanning for you too: Better to practice avoiding them before you reach ancestral space. Embers guide you, <first>!"`
```

		give ship Peregrine "Shadow Flight"
		event "remnant: cognizance peregrine"
	on decline
		dialog
			`	You have declined an essential Remnant mission, and this ends the Remnant campaign. If that was not what you intended, you may wish to reload.`

mission "Remnant: Cognizance 32"

> 	name "Pickup refined fuel."

Reviewer 3: ```suggestion
	name "Drop off refined fuel"
```

> 	description "Deliver the refined fuel to Ssil Vida by <date>."

Reviewer 3: ```suggestion
	description "Deliver the refined fuel to <destination> by <date>."
```

	landing
	deadline 19
	cargo "Refined Fuel" 58
	illegal 0
	stealth
	source "Clink"
	destination "Ssil Vida"
	to offer
		has "Remnant: Cognizance 31: done"
	on offer
		conversation

> 			`Your reception is as friendly as Clink's reputation suggests: No-one is there to meet you, and no-one arrives to help load the cargo. Just a forlorn datapad mounted on a post in front of the pile of crates requesting that you enter your pickup code to disarm the security sensors, and a statement that ships depart as soon as they are done loading. You enter the pickup code Chilia had sent you while you were in flight, load the <ship>, and get ready to head to Ssil Vida.`

Reviewer 3: ```suggestion
			`Your reception is as friendly as <origin>'s reputation suggests: No one is there to meet you, and no one arrives to help load the cargo. All that greets you is a forlorn datapad mounted on a post in front of the pile of crates, requesting that you enter your pickup code to disarm the security sensors, as well as a statement that ships depart as soon as they are done loading. You enter the pickup code Chilia had sent you while you were in flight, load the <ship>, and get ready to head to <planet>.`
```

> 			`	As you are completing your checks your sensors alert you to the arrival of a small fleet that seems to be maintaining orbit over this area. It looks like you might have to take steps to avoid observers right away.`

Reviewer 3: ```suggestion
			`	As you are completing your checks, your sensors alert you to the arrival of a small fleet that seems to be maintaining orbit over this area. It looks like you might have to take steps to avoid observers right away.`
```

				accept
	on enter "Kornephoros"
		dialog `Your comms pick up a crackling broadcast: "Don't know who you are, but there's a bounty on unaffiliated people picking up large volumes of weapons grade materials that I intend to collect!"`
	npc evade
		government "Bounty Hunter"
		personality waiting unconstrained surveillance
		system "Kornephoros"
		fleet
			names "bounty hunter"
			variant
				"Vanguard (Missile)"
				"Splinter" 2
				"Quicksilver (Proton)" 3
	on complete
		payment 250000
		event "remnant: ssil vida area unvisted"
		event "remnant: ssil vida activation"
		conversation

> 			`When you contact Ssil Vida to confirm where to unload your cargo you are greeted by Dusk, who directs you to a docking bay closer to the main reactor. As you open the hatch, though, it is Gavriil and a group of Remnant with lifters waiting. They greet you cheerfully, but they definitely seem in a rush to get the processed fuel unloaded and shuttled into the depths of the station. Once the last unit is off your ship Gavriil slows down enough to thank you for the prompt delivery, but apologizes as he rushes off as there is not much time left. His parting gesture suggests you talk to Dusk up in the control room.`

Reviewer 3: ```suggestion
			`When you contact <planet> to confirm where to unload your cargo you are greeted by Dusk, who directs you to a docking bay closer to the main reactor. As you open the hatch, though, it is Gavriil and a group of Remnant with lifters waiting. They welcome you cheerfully, but they definitely seem in a rush to get the processed fuel unloaded and shuttled into the depths of the station. Once the last unit is off your ship, Gavriil slows down enough to thank you for the prompt delivery, but apologizes as he rushes off as there is not much time left. His parting gesture suggests you talk to Dusk up in the control room.`
```

mission "Remnant: Cognizance 33"
	name "Observe changes in Postverta"
	description "Something is going on outside, and Dusk would like you to take a look."
	landing
	source "Ssil Vida"
	waypoint "Postverta"
	to offer
		has "Remnant: Cognizance 32: done"
	on offer
		conversation

> 			`Up in the control station Dusk and a small team of Remnant are busily working at a bunch of consoles. Alarms blink, and are quickly dealt with only to have new ones appear elsewhere. There is so much motion that at a glance it seems like there is twice as many of them. From a glance at the readings, it appears you brought the shipment of fuel just in time: the Aeon Cell is overloaded trying to take on the load from the fusion reactor, and most of their efforts have been directed towards carefully orchestrating rolling brown outs across the systems to keep everything minimally functional.`

Reviewer 3: ```suggestion
			`Up in the control station Dusk and a small team of Remnant are busily working at a bunch of consoles. Alarms blink and are quickly dealt with, only to have new ones appear elsewhere. There is so much motion that it seems like there is twice as many of them. From a glance at the readings, it appears you brought the shipment of fuel just in time: the Aeon Cell is overloaded trying to take on the load from the fusion reactor, and most of their efforts have been directed toward carefully orchestrating rolling brown outs across the systems to keep everything minimally functional.`
```

			`	Dusk turns towards you and the team with a satisfied gesture just beginning when there is a sudden shift as if everything had just went sideways. You almost feel like you can see a brief ripple, but it vanishes before you can focus on it. You are left with a feeling that something has changed. At the same time, the proximity sensors and systems monitoring the surrounding system light up. Someone looks up from a console. "The system is alive, Dusk! All those ghost life signs we were picking up just became much more solid!" Another gestures in clipped, fast tones. "And we have several large masses that have appeared as well. They look to be planetary in scale, but the readings seem to indicate something a bit more nebulous."`
			`	Dusk looks around, and then gestures. "Captain <first>, please go investigate what is going on out there. Just take a good look around and then report back."`
				accept
	on enter "Postverta"
		conversation

> 			`Just like the Remnant said, the system appears to be full of life. Where once there was just asteroids and a fairly empty system, there is now life signs everywhere. As you dodge amongst the asteroids, you note that most of them appear to have stuff growing on them. Plants, for lack of a better word. Moss or fungus might be a better word, although you are fairly certain that nothing in the current classifications of life can adequately categorize what you can see outside. The sudden appearance of all this life is only compounded when your sensors light up to notify you that it is tracking several flocks of void sprites.`

Reviewer 3: ```suggestion
			`Just like the Remnant said, the system appears to be full of life. Where once there was just asteroids and a fairly empty system, there is now life signs everywhere. As you dodge among the asteroids, you note that most of them appear to have something growing on them: plants. Moss or fungus might be a better word, although you are fairly certain that nothing in the current classifications of life can adequately categorize what you can see outside. The sudden appearance of all this life is only compounded when your sensors light up to notify you that it is tracking several flocks of void sprites.`
```

> 			`	As you move away from the station you spot signs of life and activity everywhere: Flitting among the rocks, soaring along the solar winds, and hunting amongst clouds. What is unclear, however, is where it came from. The only hint might be in the fact that your ship instruments report a sudden spike in the ambient energy in the surrounding area.`

Reviewer 3: ```suggestion
			`	As you move away from the station you spot signs of life and activity everywhere: flitting among the rocks, soaring along the solar winds, and hunting amongst clouds. What is unclear, however, is where it came from. The only hint might be in the fact that your ship instruments report a sudden spike in the ambient energy in the surrounding area.`
```

	on complete
		conversation

> 			`Back on the station Dusk, Gavriil, and many of the Remnant on the station meet you in the docking bay. Dusk and several of the others just came back from walking out on the hull of Ssil Vida, and they report seeing void sprites passing close to the station. They quickly pull the data from your ship sensors and throw it up on a holographic display. Everyone gathers around, intently drinking in the data and footage. They stand, stare, and ponder with intense concentration as the data continues to stream in from the station.`

Reviewer 3: ```suggestion
			`Back on the station Dusk, Gavriil, and many of the Remnant on the station meet you in the docking bay. Dusk and several of the others just came back from walking out on the hull of <origin>, and they report seeing void sprites passing close to the station. They quickly pull the data from your ship sensors and throw it up on a holographic display. Everyone gathers around, intently drinking in the data and footage. They stand, stare, and ponder with intense concentration as the data continues to stream in from the station.`
```

> 			`	Without warning, something clicks in your mind. Suddenly there is a hum of hundreds of conversations crowding your thoughts and a momentary feeling of seeing the system through thousands of eyes. You feel a wave of curiosity, and a glimpse of Ssil Vida as it might have been once - shiny and gleaming. The image ages like a fast-forward video of untold thousands of orbits as the station wears and the sky changes around it. Then the image freezes on what appears to be the present day. A feeling of appreciation replaces the sense of curiosity, followed by a blurry image of the <ship> leaving the station and heading in a dozen different directions simultaneously, as if watching it splinter into ghosts going different directions.`

Reviewer 3: ```suggestion
			`	Without warning, something clicks in your mind. Suddenly there is a hum of hundreds of conversations crowding your thoughts and a momentary feeling of seeing the system through thousands of eyes. You feel a wave of curiosity, and a glimpse of <origin> as it might have been once - shiny and gleaming. The image ages like a fast-forward video of untold thousands of orbits as the station wears and the sky changes around it. Then the image freezes on what appears to be the present day. A feeling of appreciation replaces the sense of curiosity, followed by a blurry image of the <ship> leaving the station and heading in a dozen different directions simultaneously, as if watching it splinter into ghosts going different directions.`
```

> 			`	The final image is of the creature the Remnant termed the 'Ember Node,' floating in space. The image splinters the same way as the image of your ship, and fragments into multiple ghosts of itself that vanish into a rift of energy. The sense of curiosity returns, and then there is a another click, and suddenly your mind is your own once again.`

Reviewer 3: ```suggestion
			`	The final image is of the creature the Remnant termed the "Ember Node," floating in space. The image splinters the same way as the image of your ship, and fragments into multiple ghosts of itself that vanish into a rift of energy. The sense of curiosity returns, and then there is a another click, and suddenly your mind is your own once again.`
```

mission "Remnant: Cognizance 34"
	name "Postverta system sweep"

> 	description "Conduct a sweep of nearby systems and return to Ssil Vida."

Reviewer 3: ```suggestion
	description "Conduct a sweep of nearby systems and return to <origin>."
```

	landing
	source "Ssil Vida"
	waypoint "Diespiter"
	waypoint "Statina"
	waypoint "Vaticanus"
	waypoint "Egeria"
	waypoint "Prosa"
	to offer
		has "Remnant: Cognizance 33: done"
	on offer
		conversation
			`The Remnant have a shocked expression, and a couple of them are looking dazed. Most, however, seem to be handling the incursion fairly well. Dusk gestures eagerly to you. "Good to see you recovered quickly, <first>. We will take care of everyone here - we need you out there. Go do a sweep through each of the surrounding systems, and see if you can get a signal to any of the Remnant worlds. Our communicators have all lost connection, as have the isolated backups. So we need to re-establish communication promptly." He shakes his head. "I love meeting new beings, but they always manage to surprise."`
				accept
	on waypoint
		dialog

> 			`You have successfully visited all the nearby systems. Time to report back to Ssil Vida.`

Reviewer 3: ```suggestion
			`You have successfully visited all the nearby systems. Time to report back to <origin>.`
```

			`	"Everything looks in perfect working order. The only problem is that the quantum pairs at the heart just do not seem to have a pair anymore. In terms of quantum mechanics, it seems that the rest of the universe quite simply does not exist."`
			choice
				`	"That seems problematic. How do we get back?"`
					goto cutpower
				`	"Did we destroy the universe by powering up this station?"`
			`	Gavrill smiles with amusement. "It is certainly a possibility. However, the fact that it was, as best we can tell, powered up previously without destroying the universe as far as we can tell suggests that it transported us rather than destroy everything else."`
			`	He frowns at the communication panel. "Although everything I know about quantum mechanics refuses to countenance something being able to leave the universe so absolutely like this." He smiles again. "But that is the way of science! Finding new things in the universe that tells us that what we know is not so absolute."`
			choice
				`	"I think we should focus on getting back."`
					goto cutpower
				`	"Should we be worried?"`

> 			`	Gavriil gestures neutrally "Well, I am worried, of course. I would be devastated to find out I had destroyed humanity, and everything else in existence." He looks somber, but then perks up again. "But if everything has been annihilated, there is nothing I can do about it. So I shall focus on learning what I can, doing what must be done, and enjoy the fact that the universe has inexplicably revealed once again that the sum of all our knowledge is merely the introductory metaphor to help us grasp how things work." His gestures abruptly change tone. "Speaking of which, we need to report what we have learned to the rest, which means leaving this place."`

Reviewer 3: ```suggestion
			`	Gavriil gestures neutrally. "Well, I am worried, of course. I would be devastated to find out I had destroyed humanity, and everything else in existence." He looks somber, but then perks up again. "But if everything has been annihilated, there is nothing I can do about it. So I shall focus on learning what I can, doing what must be done, and enjoy the fact that the universe has inexplicably revealed once again that the sum of all our knowledge is merely the introductory metaphor to help us grasp how things work." His gestures abruptly change tone. "Speaking of which, we need to report what we have learned to the rest, which means leaving this place."`
```
`enjoy the fact that the universe has inexplicably revealed once again that the sum of all our knowledge is merely the introductory metaphor to help us grasp how things work.`
Really like that line. Nicely done!

			label cutpower
			`	Gavriil gestures at the control panel in front of him. "I do not know for sure, but based on events, I suspect that if we manually induce a power failure we should be able to reverse the process that transported us to this realm." He smiles eagerly. "I have everything all set up, and we have people waiting across the station." He looks confident. "We think that we can safely spin down the reactor and cut power across a number of these arrays in a controlled fashion. This should replicate the original power failure, except with fewer risks and less damage to the system."`
			`	He leads you over to a panel in the main loading dock. "I know, it says 'Job Board' on it. I re-used the coding for our task list to improvise this mass-notification system." He smiles in amusement. "When you are ready to go, just select the 'De-activate Array' option from the panel. That will notify everyone and begin the shutdown."`

> 			`	He hands you a datacrystal, and then a large case. "Here is an encrypted datacrystal of all our data so far. The case has several sets, with the data stored in a variety of different mechanisms and encryptions. We do not know what happens when the station is activated or deactivated, so multiple formats for storing the data will give a better chance that at least some of it will remain intact to be delivered."`

Reviewer 3: ```suggestion
			`	He hands you a data crystal, and then a large case. "Here is an encrypted data crystal of all our data so far. The case has several sets, with the data stored in a variety of different mechanisms and encryptions. We do not know what happens when the station is activated or deactivated, so multiple formats for storing the data will give a better chance that at least some of it will remain intact to be delivered."`
```

	to offer
		has "Remnant: Cognizance 34: done"
		has "Remnant: Ssil Vida Active"
	on accept
		event "remnant: ssil vida deactivation"
		conversation
			`With a shudder that is felt in every fiber of your being, you glimpse the ripple that makes reality look like the surface of a pond after a single rain drop. Almost before it begins, everything is still again. You look at your sensors, and it looks like Postverta has once again returned to its prior, barren, form.`
	on enter Postverta
		fail

mission "Remnant: Cognizance 36"
	name "Activate Ssil Vida Power"
	description "Accept this to signal everyone to restore the power."
	job
	repeat
	source "Ssil Vida"
	to offer
		has "Remnant: Cognizance 34: done"
		not "Remnant: Ssil Vida Active"
	on accept
		event "remnant: ssil vida activation"
		conversation
			`With a shudder that is felt in every fiber of your being, you glimpse the ripple that makes reality look like the surface of a pond after a single rain drop. Almost before it begins, everything is still again. You look at your sensors, and it looks like Postverta is once again overflowing with life.`
	on enter Postverta
		fail

mission "Remnant: Cognizance 37"

> 	name "Take data to Viminal."

Reviewer 3: ```suggestion
	name "Take data to <planet>"
```

> 	description "The rest of the Remnant need to know what is happening. Take this data to them."

Reviewer 3: ```suggestion
	description "The rest of the Remnant need to know what is happening. Take this data to them on <destination>."
```

			choice
				`	(Explain everything that happened.)`
					goto cognizanceexplanation
				`	"I think I should tell this to the prefects first."`
			`	The Remnant around you look amused. "Do not be concerned, <first>. Everything you say will reach them. The sooner you tell us, the sooner we can take start preparing for whatever is necessary.`
			choice
				`	(Explain everything that happened.)`
					goto cognizanceexplanation
				`	"I thought the prefects were your leaders?"`
			` "They bear our executive responsibilities, it is true. Times of critical threats require the capacity for decisions to be made in the instant. But do not confuse the authority to make high-value decisions with being in charge of everything. The less immediate a decision is, the more likely it is to be the product of significant discussion among us all. Likewise, decisions about what constitute the policy and values of our society are things decided upon by the unified whole, not any one individual." She pauses, and smiles. "Admittedly, given your involvement in smaller projects and unfolding events, our society probably does have a certain autocratic appearance."`
			choice

> 				`	"OK, I will explain everything."`

Reviewer 3: ```suggestion
				`	"Okay, I will explain everything."`
```

					goto cognizanceexplanation
				`	"How can you reach consensus so easily?"`
			branch cognizanceuntrusted
				has "remnant untrusted"

> 			`	She looks around with an uncertain expression, then nods. "As I am sure has occurred to you, communication by gestures has some disadvantages. We still prefer it for some tasks, but it can be awkward to watch or gesture in the midst of some situations. While we have some vocabulary we can do without needing hands or arms, it is very limited.`

Reviewer 3: ```suggestion
			`	She looks around with an uncertain expression, then nods. "As I am sure has occurred to you, communication by gestures has some disadvantages. We still prefer it for some tasks, but it can be awkward to watch or gesture in the midst of some situations. While we have some vocabulary we can use without needing hands or arms, it is very limited.`
```

> 			`	"Fortunately, one of the original abilities of our companions is related to the nervous system. I believe Dawn may have told you a little about it, but was scarce on details." She smiles wryly, then another Remnant takes up the conversation. "To put it simply, we can form ad-hoc neural networks between us, of varying levels of intensity. Without the necessity of speech or gestures, we can have entire conversations in the blink of an eye."`

Reviewer 3: ```suggestion
			`	"Fortunately, one of the original abilities of our companions is related to the nervous system. I believe Dawn may have told you a little about it, but was scarce on details." She smiles wryly, then another Remnant takes up the conversation.`
			`	"To put it simply, we can form ad-hoc neural networks between us, of varying levels of intensity. Without the necessity of speech or gestures, we can have entire conversations in the blink of an eye."`
```

			`	"One such benefit is that in a policy debate, each person can be heard in depth, and includes checking the veracity of data and sources. This has significant benefits for minimizing the numbers of decisions made with incomplete or erroneous data, and maximizes the amount of information available to any one of us."`
			action
				set "Remnant: Cognizance Neural Communication Explanation"
			`	A third Remnant steps up. "As to the matter at hand, this means that the sooner any of us hear your report, the sooner everyone will have heard it." They look at you expectantly.`
				goto cognizanceexplanation
			label cognizanceuntrusted
			`	The Remnant smiles mysteriously. "A society has to have some secrets, does it not? Let us just say that our communication networks are very effective. At this point, just know that the sooner you give the report, the sooner that everyone will be able to work on this information." They pause, then continue. "And that does, of course, includes the prefects."`
			choice

> 				`	"OK, I will explain what happened."`

Reviewer 3: ```suggestion
				`	"Okay, I will explain what happened."`
```

			branch trusted
				and
					has "license: Remnant Capital"
					or
						not "Deep: Remnant 0: offered"
						has "Deep: Remnant 0: declined"
						has "Deep: Remnant 3: Secret: done"
			label surfacevault
			`	The Remnant leads you through several sets of increasingly impressive doors before arriving at a room filled with tiny doors. He picks one, uses a key to open it, and pull out a small metal box that they unlock with a second key. "If you would place the data crystal here, we will record the surface metadata, and those who need it will be able to retrieve it when necessary." He waits until you have placed it in the box, then reverses the process to secure it in the vault wall again.`
				goto departure
			label trusted

> 			`	The Remnant gestures, "Would you prefer to put it in the surface vault or the deep vault?"`

Reviewer 3: ```suggestion
			`	The Remnant gestures, "If you are in a rush, you can drop it off in the surface vault for now, and I will move it to its proper storage location myself later. Or would you like to deliver it to the deep vault yourself?"`
```

			`	You find Gavriil nearby, where he is going over a freshly installed hydroponics array with a pair of Remnant in tow. He quickly finishes and gestures farewell to the other two before walking over to you. "Greetings, Captain <first>! Welcome back to Esquiline. It has changed a bit since you were here last." His gestures have an overtone of pride. "Dusk told me that we have a new station to bring online again, so we should probably get moving.`
	on complete
		payment 250000
		conversation
			`	The equipment is quickly unloaded as Gavriil quickly scouts some of the nearby rooms. "I have picked a suitable area to start in. If you could help move these over to that room there" - he gestures towards one of the large rooms with the platforms in it - "it would be much appreciated."`
			choice
				`	"I would rather not."`
				`	"Sure."`
					goto sure
				`	"Why the rush?"`
					goto rush
			`	"Suit yourself, I have work to do." He quickly sets to moving the equipment to the desired room.`
				goto gavriilend
			label sure
			`	"Thanks, many hands make light work." You quickly have all the equipment moved into the desired room.`
				goto gavriilend
			label rush

> 			`	"Here, help me carry this and we can talk while we shift this." He hands you a lift and the two of you start moving the equipment. "Basically, we had a visitor out here. Not sure who they were, but they flew a Korath Raider with a wide mix of alien technology. Our scans said there was one Korath in charge, with a crew composed of both aliens and humans. And they came with enough firepower to make life miserable for everyone. We only had a few scouts in the area, so we stayed hidden and bargained. In the end, they gave us some 'inconsequential' data, and they got an hour to explore the station and take a few loose items. Our scans indicated that they took some stuff without much obvious value from these big halls." He gestures to the room around you with all the platforms.`

Reviewer 3: ```suggestion
			`	"Here, help me carry this and we can talk while we shift this." They hand you a lift and the two of you start moving the equipment. "Basically, we had a visitor out here. Not sure who they were, but they flew a Korath Raider with a wide mix of alien technology. Our scans said there was one Korath in charge, with a crew composed of both aliens and humans. And they came with enough firepower to make life miserable for everyone. We only had a few scouts in the area, so we stayed hidden and bargained. In the end, they gave us some 'inconsequential' data, and they got an hour to explore the station and take a few loose items. Our scans indicated that they took some stuff without much obvious value from these big halls." They gesture to the room around you with all the platforms.`
```

			choice
				`	"That is odd."`
					goto movingon
				`	"What did the Remnant get?"`
					goto remnantgain
				`	"What leverage did the Remnant use to bargain?"`

> 			`	Gavriil smiles. "Well, it seems that the captain of that ship found it distressing that we could pinpoint and detail them and their crew with sufficient data that we could use their own appearance as fake video in our transmissions, not to mention that we successfully identified which of the people on board the ship were them. And it was doubly distressing for them that, despite their impressive scanner arrays, they were not able to even catch the slightest emission that might hint at where or who we were." He pauses as he palms open a door to pull the loaded lift through, then continues. "They blustered initially, but when we started listing off precise defects and wear present on their various computer systems they decided it was better to make a deal."`

Reviewer 3: ```suggestion
			`	Gavriil smiles. "Well, it seems that the captain of that ship found it distressing that we could pinpoint and detail them and their crew with sufficient data that we could use their own appearance as fake video in our transmissions, not to mention that we successfully identified which of the people on board the ship were them. And it was doubly distressing for them that, despite their impressive scanner arrays, they were not able to even catch the slightest emission that might hint at where or who we were." Gavriil pauses as they palm open a door to pull the loaded lift through, then continues. "They blustered initially, but when we started listing off precise defects and wear present on their various computer systems they decided it was better to make a deal."`
```

			choice
				`	"Clever!"`
					goto movingon
				`	"What did you get from them?"`
			label remnantgain
			`	You finish unloading the equipment into the room before Gavriil continues. "Well, the Korath have been around for a comparatively long time. So we asked for their astronomical dataset. The data we received was raw, uncorrected, and continuous data covering multiple tens of thousands of years of stellar observations. The data even includes data that likely corresponds to the destruction of some of their worlds during their civil war." He pauses, then continues. "Of course, we need to verify the data first. So far, though, it is checking out successfully against trusted data. If it can be validated, the data will massively expand the dataset our researchers can use to model and study the universe."`
			label movingon
			`	Gavriil tucks the lifter away into a storage alcove. "Well, thank you for your help, <first>. That definitely sped up the process."`
			label gavriilend

> 			`	He looks around at the stacks of equipment, and turns to you. "This will keep me busy for a few days, but if you would be willing to do some patrols, it would be much appreciated. I will have details for you once I have a few things unpacked."`

Reviewer 3: ```suggestion
			`	You finish unloading the equipment into the room before Gavriil continues. "Well, the Korath have been around for a comparatively long time. So we asked for their astronomical dataset. The data we received was raw, uncorrected, and continuous data covering multiple tens of thousands of years of stellar observations. The data even includes data that likely corresponds to the destruction of some of their worlds during their civil war." They pause, then continue. "Of course, we need to verify the data first. So far, though, it is checking out successfully against trusted data. If it can be validated, the data will massively expand the dataset our researchers can use to model and study the universe."`
			label movingon
			`	Gavriil tucks the lifter away into a storage alcove. "Well, thank you for your help, <first>. That definitely sped up the process."`
			label gavriilend
			`	They look around at the stacks of equipment, and turn to you. "This will keep me busy for a few days, but if you would be willing to do some patrols, it would be much appreciated. I will have details for you once I have a few things unpacked."`
```

mission "Remnant: Cognizance 25"
	name "Patrol systems near Postverta."
	description "Gavriil has asked you to do a patrol through the nearby systems."
	landing
	source "Ssil Vida"
	waypoint "Diespiter"
	waypoint "Statina"
	waypoint "Egeria"
	waypoint "Vaticanus"
	waypoint "Prosa"
	to offer
		has "Remnant: Cognizance 24: done"
	on offer
		event "remnant: ssil vida inhabited"
		conversation

> 			`You have barely had a chance to check to finish your pre-flight checks on <ship> before Gavriil is tapping on your airlock. "Setting up a new environment is always an interesting challenge," Gavriil gestures. "But to the task at hand: Could you do a quick patrol through this collection of newly accessible systems? We have scouts en route with surveillance satellites to place in each of them, but I picked up the signature of a Korath Raider that was here recently, and I want to be sure that it left no surprises." He pauses, then makes a gesture you have come to recognize as meaning the accompanying message is intended as a summary. "So, visit each of the systems in this sector, and make sure that the raider did not leave anything behind." His gestures switch to a tone of finality. "Can you help with this?"`

Reviewer 3: ```suggestion
			`You have barely had a chance to check to finish your pre-flight checks on <ship> before Gavriil is tapping on your airlock. "Setting up a new environment is always an interesting challenge," Gavriil gestures. "But to the task at hand: Could you do a quick patrol through this collection of newly accessible systems? We have scouts en route with surveillance satellites to place in each of them, but I picked up the signature of a Korath Raider that was here recently, and I want to be sure that it left no surprises." They pause, then make a gesture you have come to recognize as meaning the accompanying message is intended as a summary. "So, visit each of the systems in this sector, and make sure that the raider did not leave anything behind." Their gestures switch to a tone of finality. "Can you help with this?"`
```

			choice
				`	"Yes."`
					accept
				`	"No."`
					decline
	npc kill
		government "Korath"
		personality target marked waiting
		system
			system "Egeria"
		fleet
			names "korath"
			cargo 3
			commodities Food Clothing Metal Plastic Medical "Heavy Metals"
			outfitters "Korath Basics" "Korath Exiles"
			variant
				"Korath Chaser" 3
		dialog "You have destroyed the ships that the raider left behind."
	on waypoint
		dialog "You have patrolled all six systems."
	on complete
		payment 250000
		conversation

> 			`Back on Ssil Vida, you check in with Gavriil and report the destruction of three Korath chasers that you located. "Three?" He signs quizzically. "That is an odd number. Raiders only carry two at most. I will report this right away. But in the meantime, come take a look at what I have set up." He leads you off towards the spaceport.`

Reviewer 3: ```suggestion
			`Back on Ssil Vida, you check in with Gavriil and report the destruction of three Korath chasers that you located. "Three?" They sign quizzically. "That is an odd number. Raiders only carry two at most. I will report this right away. But in the meantime, come take a look at what I have set up." They lead you off toward the spaceport.`
```

	deadline 14
	source "Ssil Vida"
	stopover "Viminal"
	to offer
		has "Remnant: Cognizance 25: done"
	on offer
		conversation
			`Gavriil strides into the room with confidence. "Look at this!" he gestures at the smooth sheeting covering the walls while the new equipment quietly hums away off to the side. A large panel with a diagram of the station stands out in the middle of it.`
			scene "scene/ssil vida alert hologram"
			`	"Setting up the life support systems as a self-contained environment within the station was simple enough that I did some preliminary power tracing on the station itself. It seems that the station is currently being powered by a immense fusion reactor that is clearly reaching its end of life." He pauses, then continues. "Not to mention it is basically running on the dregs of the dregs of the remains of its spent fuel. I do not know how much longer it will last, but likely no more than a few weeks. As best I can tell, it is still trying to power everything, but the main load" - he points to the glowing red sphere on the hologram - "seems to have simply shutdown once the power levels dropped below a certain threshold." He frowns at the display.`

> 			`	"I do not know what will happen to the station when the main reactor fails completely, and we are not interested in letting this station collapse before we have had the chance to thoroughly investigate it. As such, we need to take action now to give ourselves room to work." He turns towards you. "I have contacted Viminal, and they have an Aeon Cell ready for transport. Please go pick it up and transport it back here." He smiles faintly. "Oh, and it does not matter whether you transport it installed on your ship or in your cargo, but make sure it is not your primary power source: I have the tools to uninstall it from your ship and set it up to connect to the station, but I do not have the equipment to install it back into your ship." He pauses to check a readout on the display. "Remember: It is vital that I have the Aeon Cell by <date>.`

Reviewer 3: ```suggestion
			`Gavriil strides into the room with confidence. "Look at this!" they gesture at the smooth sheeting covering the walls while the new equipment quietly hums away off to the side. A large panel with a diagram of the station stands out in the middle of it.`
			scene "scene/ssil vida alert hologram"
			`	"Setting up the life support systems as a self-contained environment within the station was simple enough that I did some preliminary power tracing on the station itself. It seems that the station is currently being powered by a immense fusion reactor that is clearly reaching its end of life." They pause, then continue. "Not to mention it is basically running on the dregs of the dregs of the remains of its spent fuel. I do not know how much longer it will last, but likely no more than a few weeks. As best I can tell, it is still trying to power everything, but the main load" - they point to the glowing red sphere on the hologram - "seems to have simply shutdown once the power levels dropped below a certain threshold." They frown at the display.`
			`	"I do not know what will happen to the station when the main reactor fails completely, and we are not interested in letting this station collapse before we have had the chance to thoroughly investigate it. As such, we need to take action now to give ourselves room to work." They turn toward you. "I have contacted Viminal, and they have an Aeon Cell ready for transport. Please go pick it up and transport it back here." They smile faintly. "Oh, and it does not matter whether you transport it installed on your ship or in your cargo, but make sure it is not your primary power source: I have the tools to uninstall it from your ship and set it up to connect to the station, but I do not have the equipment to install it back into your ship." They pause to check a readout on the display. "Remember: It is vital that I have the Aeon Cell by <date>.`
```

				accept
	on stopover "Viminal"
		outfit "Aeon Cell" 1
		dialog "The Remnant have an Aeon cell ready when you land. They carefully move it to your ship, and several touch it reverently before they depart. You remind yourself to make sure that it is suitably stowed before taking off again."
	on complete
		payment 250000
		outfit "Aeon Cell" -1
		conversation
			`Gavriil has you unload the Aeon Cell into a corner of a nearby bay. "Thank you, <first>! The station is running critically low of power, and this should help stabilize it. We will eventually need a full team out here as well as some fuel to get the main reactor running properly, but first we need to figure out exactly what it uses."`

> 			`	He pulls out his commlink and checks a few things. "Go to Caelian and check in with Torza. He should have finished putting the team through their paces by the time you get there. See you soon, Captain!"`

Reviewer 3: ```suggestion
			`	They pull out their commlink and check a few things. "Go to Caelian and check in with Torza. He should have finished putting the team through their paces by the time you get there. See you soon, Captain!"`
```

				accept
	on complete
		payment 250000
		conversation
			`Gavriil is waiting at the dock when the <ship>'s hatches finish cycling. "Welcome," he gestures. "It is good to have fresh minds to share <planet> with." The Remnant onboard stream out of the ship in a rush, each carrying a heavy pack of equipment. They gesture respect, and head off in different directions. Evidently they have studied the maps prior to arrival, as they all seem to know where they are going. Several stay behind with Gavriil and help unload the rest of the equipment.`
			`	"We have made good progress since you were here last. We have managed to take over control of life support systems in several wings of rooms, and now those are run entirely from our own systems. Aside from relieving a tiny amount of load from the station's power core, it also means that the station could fail completely and we would still have safe living quarters to serve as base of operations."`
			`	He pauses, and his gestures take on a tone of disgust. "Of course, we are working to avoid that. But having multiple backup systems is the key to survival in hostile conditions."`

> 			`	He looks around at the few Remnant who have yet to vanish into the rest of the station. "Thank you for bringing more explorers." He hands you a data crystal. "I have already transmitted this data to Viminal, but I would appreciate it if you could take this to them to ensure there are no errors or data corruption in the transmission.`

Reviewer 3: ```suggestion
			`Gavriil is waiting at the dock when the <ship>'s hatches finish cycling. "Welcome," they gesture. "It is good to have fresh minds to share <planet> with." The Remnant onboard stream out of the ship in a rush, each carrying a heavy pack of equipment. They gesture respect, and head off in different directions. Evidently they have studied the maps prior to arrival, as they all seem to know where they are going. Several stay behind with Gavriil and help unload the rest of the equipment.`
			`	"We have made good progress since you were here last. We have managed to take over control of life support systems in several wings of rooms, and now those are run entirely from our own systems. Aside from relieving a tiny amount of load from the station's power core, it also means that the station could fail completely and we would still have safe living quarters to serve as base of operations."`
			`	They pause, and their gestures take on a tone of disgust. "Of course, we are working to avoid that. But having multiple backup systems is the key to survival in hostile conditions."`
			`	They look around at the few Remnant who have yet to vanish into the rest of the station. "Thank you for bringing more explorers." They hand you a data crystal. "I have already transmitted this data to Viminal, but I would appreciate it if you could take this to them to ensure there are no errors or data corruption in the transmission.`
```

			`He taps on his commlink. "We have inserted an order for the material into the hyperspace communication system under the company name 'Solar Winds Logistics Ltd.' The miners on <planet> are to have the shipment ready to load at a mining site just outside of the spaceport, far enough away to avoid the standard arrival and departure scans."`
			choice
				`	"Sounds good. Time to go."`
					goto departure
				`	"What about people seeing the ship?"`
			`	Chilia grins. "Fortunately for us, the diversity of human preference lends itself to our cause. While there may be only a handful of ships available for sale, spaceships are a surprisingly diverse and unregulated industry. As such, shipyards have carried on the tradition dating back to the earliest of vehicles: If someone has the wealth to pay for it, there is a shipwright willing to build it.`

> 			`	"In our time observing Ancestral space, we have seen replicas in flight of every spaceship that has ever graced popular culture or a media production. While many are barely able to manage interplanetary travel, a surprising number are in use as fully functional ships plying the hyperlanes. Given the similarity between our ships and those of certain productions, we are not concerned with being seen - only with being scanned."`

Author: Yeah. And justification for any other such ships we want to toss in there.

			`	As you are completing your checks, your sensors alert you to the arrival of a small fleet that seems to be maintaining orbit over this area. It looks like you might have to take steps to avoid observers right away.`
				accept
	on enter "Kornephoros"
		dialog `Your comms pick up a crackling broadcast: "Don't know who you are, but there's a bounty on unaffiliated people picking up large volumes of weapons grade materials that I intend to collect!"`
	npc evade
		government "Bounty Hunter"
		personality waiting unconstrained surveillance
		system "Kornephoros"
		fleet
			names "bounty hunter"
			variant
				"Vanguard (Missile)"
				"Splinter" 2
				"Quicksilver (Proton)" 3
	on complete
		payment 250000
		event "remnant: ssil vida area unvisted"
		event "remnant: ssil vida activation"
		conversation

> 			`When you contact <planet> to confirm where to unload your cargo you are greeted by Dusk, who directs you to a docking bay closer to the main reactor. As you open the hatch, though, it is Gavriil and a group of Remnant with lifters waiting. They welcome you cheerfully, but they definitely seem in a rush to get the processed fuel unloaded and shuttled into the depths of the station. Once the last unit is off your ship, Gavriil slows down enough to thank you for the prompt delivery, but apologizes as he rushes off as there is not much time left. His parting gesture suggests you talk to Dusk up in the control room.`

Reviewer 3: ```suggestion
			`When you contact <planet> to confirm where to unload your cargo you are greeted by Dusk, who directs you to a docking bay closer to the main reactor. As you open the hatch, though, it is Gavriil and a group of Remnant with lifters waiting. They welcome you cheerfully, but they definitely seem in a rush to get the processed fuel unloaded and shuttled into the depths of the station. Once the last unit is off your ship, Gavriil slows down enough to thank you for the prompt delivery, but apologizes as they rush off as there is not much time left. Their parting gesture suggests you talk to Dusk up in the control room.`
```

			`	Dusk looks around, and then gestures. "Captain <first>, please go investigate what is going on out there. Just take a good look around and then report back."`
				accept
	on enter "Postverta"
		conversation
			`Just like the Remnant said, the system appears to be full of life. Where once there was just asteroids and a fairly empty system, there is now life signs everywhere. As you dodge among the asteroids, you note that most of them appear to have something growing on them: plants. Moss or fungus might be a better word, although you are fairly certain that nothing in the current classifications of life can adequately categorize what you can see outside. The sudden appearance of all this life is only compounded when your sensors light up to notify you that it is tracking several flocks of void sprites.`

> 			`	As you move away from the station you spot signs of life and activity everywhere: flitting among the rocks, soaring along the solar winds, and hunting amongst clouds. What is unclear, however, is where it came from. The only hint might be in the fact that your ship instruments report a sudden spike in the ambient energy in the surrounding area.`

Author: ```suggestion
			`	As you move away from the station you continue to spot signs of life and activity everywhere: flitting among the rocks, soaring along the solar winds, and hunting amongst clouds. What is unclear, however, is where it came from. The only hint might be in the fact that your ship instruments report a sudden spike in the ambient energy in the surrounding area.`
```

			`	He quickly continues. "Of course, we can see it out there. But it is like looking through a very thin film." He looks thoughtful. "The current leading theory is that we are in some kind of dimensional bubble. A pocket dimension of sorts. But we have not yet reached a consensus."`
			choice
				`	"What happened to us? Did we all see the same thing?"`
				`	"Where did those feelings and images come from?"`
			`	Gavriil frowns in thought. "Based on our initial survey, each person had a slightly different experience. We think that everyone had basically the same event, but as it was a series of impressions, images, and feelings, each person picked up on different aspects of it."`

> 			`	He gestures take on a confident tone. "Which makes sense, since our life experiences shape our perceptions. Still, as to what it was..." He pauses, gathering his thoughts. "Our best guess is that this was first contact with some sort of entity." He grins at that. "I know, stating the obvious. But anyway, Dusk thinks that it might be some sort of solitary entity, as it did not use words or meanings, despite having access to our minds. As such, it uses internals to convey meaning instead of words." He glances at a map of the system. "There is, however, no indication as to where it came from.`

Reviewer 3: ```suggestion
			`Back in the control room the Remnant appear to have regained their composure. Remnant move quickly from station to station, and every sensor and tool is in use. It is as if the Remnant are afraid everything is going to vanish any moment. Which, you realize, might actually happen. You find Gavriil carefully tending a console, and they look up as you approach. "The Embers guided you back, I see. How does the outside world fare?" He listens intently as you described your flight through the surrounding systems, and nods when you mention that your navigation system is reporting no connections to the rest of the Ember Waste. Even predictions as to jump drive performance indicate no valid jumps. "No connections whatsoever?" They look thoughtful. "That actually makes sense." They gesture at a display of the area. "As best our sensors can pick up, the universe beyond this pocket of the Ember Waste just does not exist."`
			`	They quickly continue. "Of course, we can see it out there. But it is like looking through a very thin film." He looks thoughtful. "The current leading theory is that we are in some kind of dimensional bubble. A pocket dimension of sorts. But we have not yet reached a consensus."`
			choice
				`	"What happened to us? Did we all see the same thing?"`
				`	"Where did those feelings and images come from?"`
			`	Gavriil frowns in thought. "Based on our initial survey, each person had a slightly different experience. We think that everyone had basically the same event, but as it was a series of impressions, images, and feelings, each person picked up on different aspects of it."`
			`	Their gestures take on a confident tone. "Which makes sense, since our life experiences shape our perceptions. Still, as to what it was..." They pause, gathering their thoughts. "Our best guess is that this was first contact with some sort of entity." They grin at that. "I know, stating the obvious. But anyway, Dusk thinks that it might be some sort of solitary entity, as it did not use words or meanings, despite having access to our minds. As such, it uses internals to convey meaning instead of words." They glance at a map of the system. "There is, however, no indication as to where it came from.`
```

			choice
				`	"How do we get back?"`
					goto cutpower
				`	"What happened to the long range communication?"`

> 			`	"Well, that is a good question. We have, and still are, running diagnostics on them, and so far they are passing all their checks. The system safeties, isolation mechanisms, security monitoring, tamper-proofing, self-destruct systems, backup hardening..." He shakes his head, stymied.`

Reviewer 3: ```suggestion
			`	"Well, that is a good question. We have, and still are, running diagnostics on them, and so far they are passing all their checks. The system safeties, isolation mechanisms, security monitoring, tamper-proofing, self-destruct systems, backup hardening..." They shake their head, stymied.`
```

			`	"Everything looks in perfect working order. The only problem is that the quantum pairs at the heart just do not seem to have a pair anymore. In terms of quantum mechanics, it seems that the rest of the universe quite simply does not exist."`
			choice
				`	"That seems problematic. How do we get back?"`
					goto cutpower
				`	"Did we destroy the universe by powering up this station?"`
			`	Gavrill smiles with amusement. "It is certainly a possibility. However, the fact that it was, as best we can tell, powered up previously without destroying the universe as far as we can tell suggests that it transported us rather than destroy everything else."`

> 			`	He frowns at the communication panel. "Although everything I know about quantum mechanics refuses to countenance something being able to leave the universe so absolutely like this." He smiles again. "But that is the way of science! Finding new things in the universe that tells us that what we know is not so absolute."`

Reviewer 3: ```suggestion
			`	They frown at the communication panel. "Although everything I know about quantum mechanics refuses to countenance something being able to leave the universe so absolutely like this." They smile again. "But that is the way of science! Finding new things in the universe that tells us that what we know is not so absolute."`
```

			`	He leads you over to a panel in the main loading dock. "I know, it says 'Job Board' on it. I re-used the coding for our task list to improvise this mass-notification system." He smiles in amusement. "When you are ready to go, just select the 'De-activate Array' option from the panel. That will notify everyone and begin the shutdown."`
			`	He hands you a data crystal, and then a large case. "Here is an encrypted data crystal of all our data so far. The case has several sets, with the data stored in a variety of different mechanisms and encryptions. We do not know what happens when the station is activated or deactivated, so multiple formats for storing the data will give a better chance that at least some of it will remain intact to be delivered."`

> 			`	He steps back. "Well, I will return to the controls. Just give the signal when you are ready to leave. Embers light your path, Captain <first>."`

Reviewer 3: ```suggestion
			`	Gavriil gestures neutrally. "Well, I am worried, of course. I would be devastated to find out I had destroyed humanity, and everything else in existence." They look somber, but then perk up again. "But if everything has been annihilated, there is nothing I can do about it. So I shall focus on learning what I can, doing what must be done, and enjoy the fact that the universe has inexplicably revealed once again that the sum of all our knowledge is merely the introductory metaphor to help us grasp how things work." Their gestures abruptly change tone. "Speaking of which, we need to report what we have learned to the rest, which means leaving this place."`
			label cutpower
			`	Gavriil gestures at the control panel in front of them. "I do not know for sure, but based on events, I suspect that if we manually induce a power failure we should be able to reverse the process that transported us to this realm." They smile eagerly. "I have everything all set up, and we have people waiting across the station." They look confident. "We think that we can safely spin down the reactor and cut power across a number of these arrays in a controlled fashion. This should replicate the original power failure, except with fewer risks and less damage to the system."`
			`	They leads you over to a panel in the main loading dock. "I know, it says 'Job Board' on it. I re-used the coding for our task list to improvise this mass-notification system." They smile in amusement. "When you are ready to go, just select the 'De-activate Array' option from the panel. That will notify everyone and begin the shutdown."`
			`	They hand you a data crystal, and then a large case. "Here is an encrypted data crystal of all our data so far. The case has several sets, with the data stored in a variety of different mechanisms and encryptions. We do not know what happens when the station is activated or deactivated, so multiple formats for storing the data will give a better chance that at least some of it will remain intact to be delivered."`
			`	They step back. "Well, I will return to the controls. Just give the signal when you are ready to leave. Embers light your path, Captain <first>."`
```

			`	"Fortunately, one of the original abilities of our companions is related to the nervous system. I believe Dawn may have told you a little about it, but was scarce on details." She smiles wryly, then another Remnant takes up the conversation.`
			`	"To put it simply, we can form ad-hoc neural networks between us, of varying levels of intensity. Without the necessity of speech or gestures, we can have entire conversations in the blink of an eye."`
			`	He gestures confidently. "A team of Remnant, for instance, could all review incoming data from an experiment, form their own analysis, and then hold what amounts to a conference where each fully presents their thoughts and ideas with supporting data and reasoning within the span of a few minutes."`
			`	His gestures take on a cautionary tone. "Of course, accelerating to that level takes effort, and it is a long process to learn to manage our thoughts in such a way as to have productive conversations. But the results speak for themselves." He gestures broadly at the group.`

> 			`	"One such benefit is that in a policy debate, each person can be heard in depth, and includes checking the veracity of data and sources. This has significant benefits for minimizing the numbers of decisions made with incomplete or erroneous data, and maximizes the amount of information available to any one of us."`

Reviewer 6: ```suggestion
			`	"One such benefit is that in a policy debate, each person can be heard in depth, including checking the veracity of data and sources. This has significant benefits for minimizing the numbers of decisions made with incomplete or erroneous data, and maximizes the amount of information available to any one of us."`
```

			action
				set "Remnant: Cognizance Neural Communication Explanation"
			`	A third Remnant steps up. "As to the matter at hand, this means that the sooner any of us hear your report, the sooner everyone will have heard it." They look at you expectantly.`
				goto cognizanceexplanation
			label cognizanceuntrusted

> 			`	The Remnant smiles mysteriously. "A society has to have some secrets, does it not? Let us just say that our communication networks are very effective. At this point, just know that the sooner you give the report, the sooner that everyone will be able to work on this information." They pause, then continue. "And that does, of course, includes the prefects."`

Reviewer 6: ```suggestion
			`	The Remnant smiles mysteriously. "A society has to have some secrets, does it not? Let us just say that our communication networks are very effective. At this point, just know that the sooner you give the report, the sooner that everyone will be able to work on this information." They pause, then continue. "And that does, of course, include the prefects."`
```

			choice
				`	"Okay, I will explain what happened."`
				`	"Alright, you convinced me."`
			label cognizanceexplanation

> 			`	The Remnant watch you attentively as you explain everything that went on at Postverta and the surrounding area. Once you hand over the data crystals, one Remnant slots it into a console on the spot while another pair take the case and hustle off toward the information desk.`

Reviewer 6: ```suggestion
			`	The Remnant watch you attentively as you explain everything that went on at Postverta and the surrounding area. Once you hand over the data crystals, one Remnant slots one of them into a console on the spot while another pair take the case with the rest and hustle off toward the information desk.`
```

			`	He then walks briskly across the hall, gesturing you to follow. Shields flare in and out of existence around the two of you, forming a series of openings that allow you to approach the vault door, which opens when you reach it. Inside is a disappointingly small room with a small tray on a pedestal.`
			`	"If you would place the data crystal here, we will record the surface metadata, and those who need it will be able to retrieve it when necessary." He gestures to the tray, and once you place the datacrystal on it, a shield springs into existence surrounding the tray, and the whole assembly sinks into the floor.`
			choice
				`	"Well, that is done. Taely is waiting for me."`
					goto departure
				`	"Those turrets looked different"`

> 			`	The Remnant nods. "You are observant. Most of those turrets are, in fact, the originals that we discovered here. Each one is a veritable fortress in and of itself, which is why we built similar ones to use as point defence systems for our ships.`

Reviewer 7: ```suggestion
			`	The Remnant nods. "You are observant. Most of those turrets are, in fact, the originals that we discovered here. Each one is a veritable fortress in and of itself, which is why we built similar ones to use as point defense systems for our ships.`
```

			choice
				`	"Originals?"`
				`	"You did not build the vault?"`
					goto buildvault
				`	"What do you mean by fortress?"`
					goto fortress
			`	"Yes, originals. As in, they were built by those who lived here before us, and survived countless eons protecting these hidden vaults. Even after so much time, they were still sufficiently functional to be formidable obstacles. We have since repaired them and learned much, including a lot that we have yet to be able to put into practice." He sighs, but makes a gesture of anticipation. "If we can master even half the science that made these, we will be able to achieve things that would seem like magic to those still on the ancestral worlds."`
			choice
				`	"But you produce these turrets for use on ships..."`

> 				`	"What about the Point Defence Turrets on your ships?"`

Reviewer 7: ```suggestion
				`	"What about the Point Defense Turrets on your ships?"`
```

> 			`	"There is a big difference between being able to make something that looks similar, and being able to recreate something perfectly. The elements that we have formed into the defence turrets are actually the simplest and least advanced pieces of these ancient protectors." He shakes his head. "Someday, though. We are making new breakthroughs all the time."`

Reviewer 7: ```suggestion
			`	"There is a big difference between being able to make something that looks similar, and being able to recreate something perfectly. The elements that we have formed into the defense turrets are actually the simplest and least advanced pieces of these ancient protectors." He shakes his head. "Someday, though. We are making new breakthroughs all the time."`
```

				goto departure
			label fortress

> 			`	The Remnant's gestures take on a tone of curiosity. "We are not sure what they are intended to hold, but their design includes a lot of passageways and chambers that resemble a hive or a miniature city. Some have speculated that the vault builders might have been miniscule beings, which would mean that these protective devices might well have housed thousands of individuals." He shakes his head. "It must have been quite the feat to survive and prosper in a world where wildlife could annihilate a village in a single footstep." After allowing you a moment to absorb the implications, he turns back to the door. "Time to go, Captain. The sentries grow twitchy when people dawdle here."`

Reviewer 7: ```suggestion
			`	The Remnant's gestures take on a tone of curiosity. "We are not sure what they are intended to hold, but their design includes a lot of passageways and chambers that resemble a hive or a miniature city. Some have speculated that the vault builders might have been minuscule beings, which would mean that these protective devices might well have housed thousands of individuals." He shakes his head. "It must have been quite the feat to survive and prosper in a world where wildlife could annihilate a village in a single footstep." After allowing you a moment to absorb the implications, he turns back to the door. "Time to go, Captain. The sentries grow twitchy when people dawdle here."`
```

				goto departure
			label buildvault

> 			`	"No, we did not build the vaults. They were built by ancient inhabitants of these worlds. We were lucky to find them, hidden away as they were. We have studied them, repaired them, and added additional layers of our own; but fundamentally these are still the structures left behind by those who came before." He nods appreciatively at the surrounding room. Taking a moment to look around, you note that the walls and floor join seemlessly, and are made of a dark metallic substance that seems to shimmer. After allowing you a moment to admire the view, he turns back to the door. "Time to go, Captain. We would not want the sentries to think you were dawdling here."`

Reviewer 7: ```suggestion
			`	"No, we did not build the vaults. They were built by ancient inhabitants of these worlds. We were lucky to find them, hidden away as they were. We have studied them, repaired them, and added additional layers of our own; but fundamentally these are still the structures left behind by those who came before." He nods appreciatively at the surrounding room. Taking a moment to look around, you note that the walls and floor join seamlessly, and are made of a dark metallic substance that seems to shimmer. After allowing you a moment to admire the view, he turns back to the door. "Time to go, Captain. We would not want the sentries to think you were dawdling here."`
```

			`	Before long the technicians and pilots are trading stories while the Mergansers overshadow the group. Sometimes they poke various engineers or technicians, and the technicians frequently share messages from their commlinks. Apparently they have them hooked up to the Mergansers, so the ships are sharing their own accounts of the patrol. After several hours of the festivities, Taely announces that everyone needs to get to sleep because tomorrow is a new day. As everyone bids their farewells, you note that they include the ships, with most people patting or stroking one or more of the ships on their way out.`

> 			`	"Stop by the cafeteria tommorrow morning, <first>. There will be more you can help with, if you are interested." With that, she gestures farewell and heads off at a brisk pace.`

Reviewer 7: ```suggestion
			`	"Stop by the cafeteria tomorrow morning, <first>. There will be more you can help with, if you are interested." With that, she gestures farewell and heads off at a brisk pace.`
```

				accept
	npc
		government "Korath"
		personality heroic plunders harvests waiting staying
		system "Parca"
		fleet
			names "korath"
			cargo 3
			commodities Food Metal Plastic "Heavy Metals"
			outfitters "Korath Basics" "Korath Exiles"
			variant
				"Korath Dredger" 1
				"Korath Dredger (Ground Assault)" 2
				"Korath Raider (Ember)" 6
				"Korath Chaser" 24

mission "Remnant: Cognizance 28"
	name "Recover equipment from Baianus."
	description "Torza has asked you to recover equipment from Baianus by <date>."
	landing
	blocked `The scene is one of organized chaos. One Remnant signs "We need a ship that can carry 280 cargo and 20 passengers."`
	deadline 12
	cargo "equipment" 280
	passengers 20
	source "Baianus"
	destination "Caelian"
	to offer
		has "Remnant: Cognizance 27: done"
	on offer
		conversation

> 			`Punching through the Korath blockade finds you racing across the aurora filled skies of Baianus. You quickly slide into the designated approach track and start to enter your IFF code as a barrage of anti-air fire erupts from the ground to lance over your shoulder. As you drop into evasive manouvers you spot the wreckage of a chaser tumbling to the ice behind you, and a firelance beam sears the air where the <ship> had just been. Moments later you get the confirmation signal that they have accepted your ID and are tracking your position.`

Reviewer 7: ```suggestion
			`Punching through the Korath blockade finds you racing across the aurora filled skies of Baianus. You quickly slide into the designated approach track and start to enter your IFF code as a barrage of anti-air fire erupts from the ground to lance over your shoulder. As you drop into evasive maneuvers you spot the wreckage of a chaser tumbling to the ice behind you, and a firelance beam sears the air where the <ship> had just been. Moments later you get the confirmation signal that they have accepted your ID and are tracking your position.`
```

			`	As you approach the entrance to the starport directions appear on your comm panel directing you to a spot where a large stack of equipment surrounds the landing pad. You ease inside and have barely touched down when a technician is dropping from the ceiling down onto your hull with a refueling hose and repair equipment. Meanwhile, a swarm of Remnant have equipment in motion. You quickly open the cargo hold doors, and the Remnant rush onboard directing lifts and loaders.`
			`	The Remnant move fast, but you can hear the battle approaching. Even through the <ship>'s dampeners you can feel the heavy thud of anti-air artillery. The whine of generators being pushed to the limit fills the air as the shields crackle with incoming fire. Through the cacophany the Remnant move with purpose, gesturing instructions silently through the din. Finally the last load is packed onboard, and one of them comes up to the cockpit. "Thank you for your assistance, Captain <last>. As the explosive charges have been set, I recommend we leave promptly." They nod respectfully, then strap themselves into the nearest harness.`

> 			`	The shield opens one final time to allow you to leave, leaping practically into the teeth of a Korath attack. The anti-air artillery continues its rythmic barrage against the incoming ships, but it is evident that both it and the shields around the base are failing under the assault. As you head for orbit, the last image you see is of the artillery pieces sag and collapse as they melt into the ground. Your sensors record a sudden explosion from the cavern, to which the Remnant give a sad but satisfied nod. "Take us back to Caelian please, Captain."`

Reviewer 7: ```suggestion
			`	The shield opens one final time to allow you to leave, leaping practically into the teeth of a Korath attack. The anti-air artillery continues its rhythmic barrage against the incoming ships, but it is evident that both it and the shields around the base are failing under the assault. As you head for orbit, the last image you see is of the artillery pieces sag and collapse as they melt into the ground. Your sensors record a sudden explosion from the cavern, to which the Remnant give a sad but satisfied nod. "Take us back to Caelian please, Captain."`
```

			choice
				`	(drink in silence)`
					goto powerbriefing
				`	(ask the Remnant next to you what just happened)`
			`	The Remnant smiles a tiredly. "We did a ten kilometer run in just under thirty seven minutes, wearing our emergency environmental suits and carrying basic survival and maintnance gear. Torza set a hard pace, but we kept up." She takes a sip from her drink. "It is hard work, but endurance is rather important for survival, especially in an unknown environment like Ssil Vida." She takes another sip. "It was an interesting changes of pace. Normally I carry full engineering kit, so this was a 'fast scramble' in comparison. He," she gestures at Torza, "likes to change things up. No two runs are the same when he sets the pace."`
				goto powerbriefing
			label notthirsty
			`	Torza quickly continues his rounds, and soon everyone is sipping on a drink. Several have pulled up displays of Ssil Vida, while others are chatting as they go over inventory lists. Most of the conversation that you catch sounds like a group of technicians prepping for a mission.`
			label powerbriefing

> 			`	After a few minutes Torza gestures for everyone to pay attention. "Now, to review the paramters: Ssil Vida appears to be a giant mechanism for doing something we do not yet know. The core systems have exhausted the capacity of its large fusion reactor to supply it. The installation of the Aeon Cell has reduced the strain on the system, but cannot fully power the station. You need to focus on the power systems, and figure out how to fully stabilize the distribution. While you are doing that, <first> will be returning to Viminal for the next step in this operation. You will all be staying on Ssil Vida for at least a few months, maybe longer as the situation warrants. Agreed?" Around the room there are a variety of nods and gestures of affirmation.`

Reviewer 7: ```suggestion
			`	After a few minutes Torza gestures for everyone to pay attention. "Now, to review the parameters: Ssil Vida appears to be a giant mechanism for doing something we do not yet know. The core systems have exhausted the capacity of its large fusion reactor to supply it. The installation of the Aeon Cell has reduced the strain on the system, but cannot fully power the station. You need to focus on the power systems, and figure out how to fully stabilize the distribution. While you are doing that, <first> will be returning to Viminal for the next step in this operation. You will all be staying on Ssil Vida for at least a few months, maybe longer as the situation warrants. Agreed?" Around the room there are a variety of nods and gestures of affirmation.`
```

			`	"Based on our analysis, the reactor appears to use a very specific isotope. Whether it is truly needed or others would suffice is unknown, but for now we are inclined to provide it with an exact match. While we will be able to produce it eventually, it will take time that we do not have." The diagram of Ssil Vida vanishes and is replaced by a map of the galaxy, which focuses in on Kornephoros, near the juncture of the Rim and the South. "Our records indicate that the most easily accessible deposits are on Clink." Chilia turns to you. "As you are from Ancestral space and are most familiar with the people there, we would like you to travel there and bring back 58 cubic meters of processed fuel for the reactor. Can you do this?"`
				goto ssilvidacontinue
			label ssilvidaemeraldsword

> 			`	A hologram of Ssil Vida appears in the air over the table, then fades away to leave a large highlighted object with a network of lines reaching around the station. "As Gavriil has told you, the station's primary power source appears to be a massive fusion reactor. Based on our intial scans, it appears to be a match to the Sheragi reactor you recently discovered, albeit on a vastly larger scale. Unfortunately, it has consumed its entire fuel supply and is currently using up the last of the fuel it is capable of extracting by refining spent fuel. Our current estimates is that it is producing less than ten percent of its nominal power. This is sufficiently low that it has already ceased powering the majority of the station's systems." He gestures to Chilia.`

Reviewer 7: ```suggestion
			`	A hologram of Ssil Vida appears in the air over the table, then fades away to leave a large highlighted object with a network of lines reaching around the station. "As Gavriil has told you, the station's primary power source appears to be a massive fusion reactor. Based on our initial scans, it appears to be a match to the Sheragi reactor you recently discovered, albeit on a vastly larger scale. Unfortunately, it has consumed its entire fuel supply and is currently using up the last of the fuel it is capable of extracting by refining spent fuel. Our current estimates is that it is producing less than ten percent of its nominal power. This is sufficiently low that it has already ceased powering the majority of the station's systems." He gestures to Chilia.`
```

			`	There, crouched next to <ship>, is what you can only describe as a predator: It has long sleek lines with flared wings that resemble feathers, and a massive engine array that suggest incomparable mobility in a ship of its size. As you approach you can see several Remnant working around it. One gestures, and the ship extends a wing and the Remnant reaches in between the feathers to do something, then steps back and the ship runs through a range of twisting motions as it moves its wings around. It strongly resembles a raptor preening its feathers.`
			`	Taely walks right up to the nose and the ship crouches down so she can put her hand on its nose. "Captain <first>, I would like to introduce you to the Peregrine. It is the first new capital warship design we have built in over a century, and is the result of decades of work on my part as well as many others. Hopefully, it is the first of many new ships that are successful."`
			choice
				`	"Looks great. Let's fly!"`
					goto letsfly
				`	"What sort of ship is it?"`

> 			`	Taely gestures in reverence. "It is what we term a liveship." She scowls at you. "And before you start thinking about ships made of big piles of flesh and ooze, this is obviously not that." She runs her hand along the ship, obviously quite smooth and solid. "Liveships, as we are calling them, are spaceships that have a sufficient degree of self-awareness and self-control to be deemed alive. They are manufactured, and have energy and fuel requirements just like any ship. Likewise, they are largely made of materials similar to what you are familiar with. But they are concious. They can move and flex in ways that would be impossible to precisely control with servos and traditional position reporting. They can feel where each tendril is as surely as most people can tell where their hand is, and each ripple of a wormhole vortex is as perceptible to them as the crest of a sand dune would be to you or I."`

Reviewer 7: ```suggestion
			`	Taely gestures in reverence. "It is what we term a liveship." She scowls at you. "And before you start thinking about ships made of big piles of flesh and ooze, this is obviously not that." She runs her hand along the ship, obviously quite smooth and solid. "Liveships, as we are calling them, are spaceships that have a sufficient degree of self-awareness and self-control to be deemed alive. They are manufactured, and have energy and fuel requirements just like any ship. Likewise, they are largely made of materials similar to what you are familiar with. But they are conscious. They can move and flex in ways that would be impossible to precisely control with servos and traditional position reporting. They can feel where each tendril is as surely as most people can tell where their hand is, and each ripple of a wormhole vortex is as perceptible to them as the crest of a sand dune would be to you or I."`
```

mission "Remnant: Cognizance 34"
	name "Postverta system sweep."
	description "Conduct a sweep of nearby systems and return to Ssil Vida."
	landing
	source "Ssil Vida"
	waypoint "Diespiter"
	waypoint "Statina"
	waypoint "Vaticanus"
	waypoint "Egeria"
	waypoint "Prosa"
	to offer
		has "Remnant: Cognizance 33: done"
	on offer
		conversation
			`The Remnant have a shocked expression, and a couple of them are look dazed. Most, however, seem to be handling the incursion fairly well. Dusk gestures eagerly to you. "Good to see you recovered quickly, <first>. We will take care of everyone here - we need you out there. Go do a sweep through each of the surrounding systems, and see if you can get a signal to any of the Remnant worlds. Our communicators have all lost connection, as have the isolated backups. So we need to re-establish communication promptly." He shakes his head. "I love meeting new beings, but they always manage to surprise."`
				accept
	on waypoint
		dialog

> 			`You have succesfully visited all the nearby systems. Time to report back to Ssil Vida.`

Reviewer 7: ```suggestion
			`You have successfully visited all the nearby systems. Time to report back to Ssil Vida.`
```

	on complete
		conversation
			`Back in the control room the Remnant appear to have regained their composure. Remnant move quickly from station to station, and every sensor and tool is in use. It is as if the Remnant are afraid everything is going to vanish any moment. Which, you realize, might actually happen. You find Gavriil carefully tending a console, and he looks up as you approach. "The Embers guided you back, I see. How does the outside world fare?" He listens intently as you described your flight through the surrounding systems, and nods when you mention that your navigation system is reporting no connections to the rest of the Ember Waste. Even predictions as to jump drive performance indicate no valid jumps. "No connections whatsoever?" He looks thoughtful. "That actually makes sense." He gestures at a display of the area. "As best our sensors can pick up, the universe beyond this pocket of the Ember Waste just does not exist."`

> 			`	He quickly continues. "Of course, we can see it out there. But it is like looking through a very thin film." He looks thoughtful. "The current leading theory is that we are in some kind of dimensional bubble. A pocket dimension of sorts. But we have not yet reached a concensus."`

Reviewer 7: ```suggestion
			`	He quickly continues. "Of course, we can see it out there. But it is like looking through a very thin film." He looks thoughtful. "The current leading theory is that we are in some kind of dimensional bubble. A pocket dimension of sorts. But we have not yet reached a consensus."`
```

			choice
				`	"That seems problematic. How do we get back?"`
					goto cutpower
				`	"Did we destroy the universe by powering up this station?"`
			`	Gavrill smiles with amusement. "It is certainly a possibility. However, the fact that it was, as best we can tell, powered up previously without destroying the universe as far as we can tell suggests that it transported us rather than destroying everything else." He frowns at the communication panel. "Although everything I know about quantum mechanics refuses to countenance something being able to leave the universe so absolutely like this." He smiles again. "But that is the way of science! Finding new things in the universe that tell us that what we know is not so absolute."`
			choice
				`	"I think we should focus on getting back."`
					goto cutpower
				`	"Should we be worried?"`

> 			`	Gavriil gestures neutrally "Well, I am worried, of course. I would be devastated to find out I had destroyed humanity, and everything else in existance." He looks somber, but then perks up again. "But if everything has been annihilated, there is nothing I can do about it. So I shall focus on learning what I can, doing what must be done, and enjoy the fact that the universe has unexplicably revealed once again that the sum of all our knowledge is merely the introductory metaphor to help us grasp how things work." His gestures abruptly change tone. "Speaking of which, we need to report what we have learned to the rest, which means leaving this place."`

Reviewer 7: ```suggestion
			`	Gavriil gestures neutrally "Well, I am worried, of course. I would be devastated to find out I had destroyed humanity, and everything else in existence." He looks somber, but then perks up again. "But if everything has been annihilated, there is nothing I can do about it. So I shall focus on learning what I can, doing what must be done, and enjoy the fact that the universe has inexplicably revealed once again that the sum of all our knowledge is merely the introductory metaphor to help us grasp how things work." His gestures abruptly change tone. "Speaking of which, we need to report what we have learned to the rest, which means leaving this place."`
```

				distance 235
				period 14
		object "Wormhole Link"
			sprite planet/wormhole-red
			distance 2754
			period 389

mission "Remnant: Cognizance 37"
	name "Take data to Viminal."
	description "The rest of the Remnant need to know what is happening. Take this data to them."
	landing
	source "Ssil Vida"
	destination "Viminal"
	to offer
		has "Remnant: Cognizance 34: done"
	on complete
		event "remnant: cognizance salvage expansion"
		conversation
			`A crowd of Remnant cluster around your ship as you land, but it is just one who gestures quickly, asking if the rest of the team who went with you are safe. You nod, and a ripple of relief goes through the crowd. "So, what happened?" asks another.`
			choice
				`	(Explain everything that happened.)`
					goto cognizanceexplanation
				`	"I think I should tell this to the prefects first."`
			`	The Remnant around you look amused. "Do not be concerned, <first>. Everything you say will reach them. The sooner you tell us, the sooner we can take start preparing for whatever is necessary.`
			choice
				`	(Explain everything that happened.)`
					goto cognizanceexplanation
				`	"I thought the prefects were your leaders?"`

> 			` "They bear our executive responsibilities, it is true. Times of critical threats require the capacity for decisions to be made in the instant. But do not confuse the authority to make high-value decisions with being in charge of everything. The less immediate a decision is, the more likely it is to be the product of significant discussion among us all. Likewise, decisions about what constitute the policy and values of our society are things decided upon by the unified whole, not any one individual." She pauses, and smiles. "Admitedly, given your involvement in smaller projects and unfolding events, our society probably does have a certain autocratic appearance."`

Reviewer 7: ```suggestion
			` "They bear our executive responsibilities, it is true. Times of critical threats require the capacity for decisions to be made in the instant. But do not confuse the authority to make high-value decisions with being in charge of everything. The less immediate a decision is, the more likely it is to be the product of significant discussion among us all. Likewise, decisions about what constitute the policy and values of our society are things decided upon by the unified whole, not any one individual." She pauses, and smiles. "Admittedly, given your involvement in smaller projects and unfolding events, our society probably does have a certain autocratic appearance."`
```

			choice
				`	"OK, I will explain everything."`
					goto cognizanceexplanation

> 				`	"How can you reach concensus so easily?"`

Reviewer 7: ```suggestion
				`	"How can you reach consensus so easily?"`
```

			branch cognizanceuntrusted
				has "remnant untrusted"

> 			`	She looks around with an uncertain expression, then nods. "As I am sure has occured to you, communication by gestures has some disadvantages. We still prefer it for some tasks, but it can be ackward to watch or gesture in the midst of some situations. While we have some vocabulary we can do without needing hands or arms, it is very limited. Fortunately, one of the original abilities of our companions is related to the nervous system. I believe Dawn may have told you a little about it, but was scarce on details." She smiles wryly, then another Remnant takes up the conversation. "To put it simply, we can form ad-hoc neural networks between us, of varying levels of intensity. Without the necessity of speech or gestures, we can have entire conversations in the blink of an eye."`

Reviewer 7: ```suggestion
			`	She looks around with an uncertain expression, then nods. "As I am sure has occurred to you, communication by gestures has some disadvantages. We still prefer it for some tasks, but it can be awkward to watch or gesture in the midst of some situations. While we have some vocabulary we can do without needing hands or arms, it is very limited. Fortunately, one of the original abilities of our companions is related to the nervous system. I believe Dawn may have told you a little about it, but was scarce on details." She smiles wryly, then another Remnant takes up the conversation. "To put it simply, we can form ad-hoc neural networks between us, of varying levels of intensity. Without the necessity of speech or gestures, we can have entire conversations in the blink of an eye."`
```

		fleet
			names "korath"
			cargo 1
			commodities "void sprite parts"
			variant
				"Korath Raider (Ember)"
				"Korath Chaser" 2
	on enter "Nenia"
		dialog
			`As you scan the system, the first thing you notice is that the void sprites' behavior seems different: There are fewer of them flying around, and the few that you see seem to be almost timid - floating quietly, and dropping down into the cloud deck whenever your ship moves towards them. You recall that Plume asked you to visit both planets.`
	on stopover
		conversation
			`	The void sprites here prove to be elusive. You spend quite a while chasing radar signatures before you finally manage to get a small flock of them into visual range. You record them from a distance for several minutes, until they suddenly disperse into a storm and vanish.`
	on complete
		conversation
			`Disappointed in the void sprites' behavior, you deliver the recordings to Plume's lab without fanfare. He quickly takes them and reviews the footage. He makes a gesture that you think means 'curious dejection', then seems to notice something, and resumes his usual intent bearing.`

> 			`	"Here, look!" He rewinds the recording and gestures curtly at the projection in front of you. "There. That one is injured. Some kind of burn in a long, straight line." He zips through the footage farther: "And here, a scattering of scorch marks." He looks up at you and snaps out a series of gestures: "This is outside my experience; I need to consult an expert. Meet me in the spaceport in an hour."`

Author: Looks like a good change. If you could just expand the contraction, I'll apply it.

		planet "Ssil Vida"
	to offer
		has "Remnant: Cognizance 20: done"
		not "Remnant: Dusk Busy"
	on offer
		require "Research Laboratory"
		conversation
			branch duskknown
				has "remnant met dusk"
			`	In the cafeteria the scene hasn't changed much: There is still a large group of Remnant gathered around a cluster of projectors, with a constant stream of gestures discussing various points of the data. You understand enough to figure out that most of the conversation is deep into technical discussions of how the lifesign sensors might give the readings they do without having anything visible. The principle differences from before is that there is a stack of small bags and helmets next to the door and five of the Remnant present are wearing pressure suits. One of them notices you and waves you over. "Captain <first>! My name is Dusk, and I am one of the researchers that you volunteered to transport to Ssil Vida."`
			choice
				`	"Pleased to meet you, Dusk."`
					goto missiondescription
				`	"Dusk? Are names like that common?"`
					goto duskname
			label duskknown

> 			`	In the cafeteria the scene hasn't changed much: There is still a large group of Remnant gathered around a cluster of projectors, with a constant stream of gestures discussing various points of the data. You understand enough to figure out that most of the conversation is deep into technical discussions of how the lifesign sensors might give the readings they do without having anything visible. The principle differences from before is that there is a stack of small bags and helmets next to the door and five of the Remnant present are wearing pressure suits. You recognize one of them as Dusk as they waves you over. "Captain <first>! The Embers crossed our paths once again. I am one of the researchers that you volunteered to transport to Ssil Vida."`

Reviewer 3: ```suggestion
			`	In the cafeteria the scene hasn't changed much: There is still a large group of Remnant gathered around a cluster of projectors, with a constant stream of gestures discussing various points of the data. You understand enough to figure out that most of the conversation is deep into technical discussions of how the lifesign sensors might give the readings they do without having anything visible. The principle differences from before is that there is a stack of small bags and helmets next to the door and five of the Remnant present are wearing pressure suits. You recognize one of them as Dusk as they waves you over. "Captain <first>! The Embers crossed our paths once again. I am one of the researchers that you have volunteered to transport to Ssil Vida."`
```

			`	Dusk leans over the holo, peering at the details. "The mission is a very standard arc: We go to the station, set up our equipment to get some detailed readings, and return here once we are done. For these initial readings we should need about a day onsite, and we will leave some of our equipment to take ongoing recordings for retrieval later. After that... Well, we will see what the data says and decide from there."`
				accept
	on accept
		set "Remnant: Dusk Busy"

> 		"remnant met dusk" ++

Reviewer 3: Why not just use below?
```suggestion
		set "remnant met dusk"
```
Granted, I'm not a wiz with using triggers. Was there something specific your method achieves that my suggestion doesn't in regards to using this in #6153 as well?

Author: The main difference here is that by using ++ it will increment upwards instead of just setting it. So it can be used to see if the player has been through more than one of these meeting later on. I do intend to make use of that later. So it is a case of "enables future branching without incurring any additional cost now"

mission "Remnant: Cognizance 27"

> 	name "Check in with Torza on Caelian"

Reviewer 3: ```suggestion
	name "Check in with Torza on <planet>"
```

> 	description "Torza should have a team ready on Caelian."

Reviewer 3: ```suggestion
	description "Torza should have a team ready on <planet>."
```

	on offer
		conversation
			`Punching through the Korath blockade finds you racing across the aurora filled skies of Baianus. You quickly slide into the designated approach track and start to enter your IFF code as a barrage of anti-air fire erupts from the ground to lance over your shoulder. As you drop into evasive maneuvers you spot the wreckage of a chaser tumbling to the ice behind you, and a firelance beam sears the air where the <ship> had just been. Moments later you get the confirmation signal that they have accepted your ID and are tracking your position.`

> 			`	As you approach the entrance to the hidden starport, directions appear on your comm panel directing you to a spot where a large stack of equipment surrounds the landing pad. You ease inside and have barely touched down when a technician is dropping from the ceiling onto your hull with a refueling hose and repair equipment. Meanwhile, a swarm of Remnant are prepping and moving various equipment, many with lifts and loaders. You quickly open the cargo hold doors, and the Remnant rush to load and board the <ship>.`

Reviewer 6: ```suggestion
			`	As you approach the entrance to the hidden starport, directions appear on your comm panel guiding you to a spot where a large stack of equipment surrounds the landing pad. You ease inside and have barely touched down when a technician is dropping from the ceiling onto your hull with a refueling hose and repair equipment. Meanwhile, a swarm of Remnant are prepping and moving various equipment, many with lifts and loaders. You quickly open the cargo hold doors, and the Remnant rush to load and board the <ship>.`
```

Missed directions directing you the first time
